% Template for PLoS
% Version 3.5 March 2018
%
% % % % % % % % % % % % % % % % % % % % % %
%
% -- IMPORTANT NOTE
%
% This template contains comments intended 
% to minimize problems and delays during our production 
% process. Please follow the template instructions
% whenever possible.
%
% % % % % % % % % % % % % % % % % % % % % % % 
%
% Once your paper is accepted for publication, 
% PLEASE REMOVE ALL TRACKED CHANGES in this file 
% and leave only the final text of your manuscript. 
% PLOS recommends the use of latexdiff to track changes during review, as this will help to maintain a clean tex file.
% Visit https://www.ctan.org/pkg/latexdiff?lang=en for info or contact us at latex@plos.org.
%
%
% There are no restrictions on package use within the LaTeX files except that 
% no packages listed in the template may be deleted.
%
% Please do not include colors or graphics in the text.
%
% The manuscript LaTeX source should be contained within a single file (do not use \input, \externaldocument, or similar commands).
%
% % % % % % % % % % % % % % % % % % % % % % %
%
% -- FIGURES AND TABLES
%
% Please include tables/figure captions directly after the paragraph where they are first cited in the text.
%
% DO NOT INCLUDE GRAPHICS IN YOUR MANUSCRIPT
% - Figures should be uploaded separately from your manuscript file. 
% - Figures generated using LaTeX should be extracted and removed from the PDF before submission. 
% - Figures containing multiple panels/subfigures must be combined into one image file before submission.
% For figure citations, please use "Fig" instead of "Figure".
% See http://journals.plos.org/plosone/s/figures for PLOS figure guidelines.
%
% Tables should be cell-based and may not contain:
% - spacing/line breaks within cells to alter layout or alignment
% - do not nest tabular environments (no tabular environments within tabular environments)
% - no graphics or colored text (cell background color/shading OK)
% See http://journals.plos.org/plosone/s/tables for table guidelines.
%
% For tables that exceed the width of the text column, use the adjustwidth environment as illustrated in the example table in text below.
%
% % % % % % % % % % % % % % % % % % % % % % % %
%
% -- EQUATIONS, MATH SYMBOLS, SUBSCRIPTS, AND SUPERSCRIPTS
%
% IMPORTANT
% Below are a few tips to help format your equations and other special characters according to our specifications. For more tips to help reduce the possibility of formatting errors during conversion, please see our LaTeX guidelines at http://journals.plos.org/plosone/s/latex
%
% For inline equations, please be sure to include all portions of an equation in the math environment.  For example, x$^2$ is incorrect; this should be formatted as $x^2$ (or $\mathrm{x}^2$ if the romanized font is desired).
%
% Do not include text that is not math in the math environment. For example, CO2 should be written as CO\textsubscript{2} instead of CO$_2$.
%
% Please add line breaks to long display equations when possible in order to fit size of the column. 
%
% For inline equations, please do not include punctuation (commas, etc) within the math environment unless this is part of the equation.
%
% When adding superscript or subscripts outside of brackets/braces, please group using {}.  For example, change "[U(D,E,\gamma)]^2" to "{[U(D,E,\gamma)]}^2". 
%
% Do not use \cal for caligraphic font.  Instead, use \mathcal{}
%
% % % % % % % % % % % % % % % % % % % % % % % % 
%
% Please contact latex@plos.org with any questions.
%
% % % % % % % % % % % % % % % % % % % % % % % %

\documentclass[10pt,letterpaper]{article}
\usepackage[top=0.85in,left=2.75in,footskip=0.75in]{geometry}

% amsmath and amssymb packages, useful for mathematical formulas and symbols
\usepackage{amsmath,amssymb}

% Use adjustwidth environment to exceed column width (see example table in text)
\usepackage{changepage}

% Use Unicode characters when possible
\usepackage[utf8x]{inputenc}

% textcomp package and marvosym package for additional characters
\usepackage{textcomp,marvosym}

% cite package, to clean up citations in the main text. Do not remove.
\usepackage{cite}

% Use nameref to cite supporting information files (see Supporting Information section for more info)
\usepackage{nameref,hyperref}

% line numbers
\usepackage[right]{lineno}

% ligatures disabled
\usepackage{microtype}
\DisableLigatures[f]{encoding = *, family = * }

% color can be used to apply background shading to table cells only
\usepackage[table]{xcolor}

% array package and thick rules for tables
\usepackage{array}

% create "+" rule type for thick vertical lines
\newcolumntype{+}{!{\vrule width 2pt}}

% create \thickcline for thick horizontal lines of variable length
\newlength\savedwidth
\newcommand\thickcline[1]{%
  \noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
  \cline{#1}%
  \noalign{\vskip\arrayrulewidth}%
  \noalign{\global\arrayrulewidth\savedwidth}%
}

% \thickhline command for thick horizontal lines that span the table
\newcommand\thickhline{\noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
\hline
\noalign{\global\arrayrulewidth\savedwidth}}

% My packages
\usepackage{xspace}
\usepackage{gensymb}
\usepackage{siunitx}
\usepackage{soul}
% My commands
\newcommand{\indep}{\rotatebox[origin=c]{90}{$\models$}}
\newcommand{\eg}{\emph{e.g.}\xspace}
\newcommand{\ie}{\emph{i.e.}\xspace}
\newcommand{\T}{^\mathrm{T}}
\newcommand{\bbeta}{\boldsymbol{\beta}}
\newcommand{\blup}{\widetilde{\bbeta}}
\newcommand{\bzero}{\mathbf{0}}
\newcommand{\bI}{\mathbf{I}}
\newcommand{\bx}{\mathbf{x}}
\newcommand{\bX}{\mathbf{X}}
\newcommand{\by}{\mathbf{y}}
\newcommand{\tausq}{\tau^{2}}
\newcommand{\qtl}{\widehat{h^{2}}_{\text{QTL}}}

\newcommand{\permpc}{\text{permP}_{\text{C}}}
\newcommand{\permpg}{\text{permP}_{\text{G}}}

\newcommand{\permpmed}{\text{permP}^{m}}

\newcommand{\WV}[2]{\textcolor{red}{#1\footnote{\textcolor{red}{WV: #2}}}}
\newcommand{\WVinline}[1]{\textcolor{red}{#1}}

\newcommand{\GK}[2]{\textcolor{teal}{#1\footnote{\textcolor{teal}{GK: #2}}}}
\newcommand{\GKinline}[1]{\textcolor{teal}{#1}}

\newcommand{\BQ}[2]{\textcolor{orange}{#1\footnote{\textcolor{orange}{BQ: #2}}}}
\newcommand{\BQinline}[1]{\textcolor{orange}{#1}}
\newcommand{\BQst}[1]{\textcolor{orange}{\st{#1}}}

\newcommand{\TF}[2]{\textcolor{purple}{#1\footnote{\textcolor{purple}{TF: #2}}}}
\newcommand{\TFinline}[1]{\textcolor{purple}{#1}}



% Remove comment for double spacing
%\usepackage{setspace} 
%\doublespacing

% Text layout
\raggedright
\setlength{\parindent}{0.5cm}
\textwidth 5.25in 
\textheight 8.75in

% Bold the 'Figure #' in the caption and separate it from the title/caption with a period
% Captions will be left justified
\usepackage[aboveskip=1pt,labelfont=bf,labelsep=period,justification=raggedright,singlelinecheck=off]{caption}
\renewcommand{\figurename}{Fig}

% Use the PLoS provided BiBTeX style
\bibliographystyle{plos2015}

% Remove brackets from numbering in List of References
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother



% Header and Footer with logo
\usepackage{lastpage,fancyhdr,graphicx}
\usepackage{epstopdf}
%\pagestyle{myheadings}
\pagestyle{fancy}
\fancyhf{}
%\setlength{\headheight}{27.023pt}
%\lhead{\includegraphics[width=2.0in]{PLOS-submission.eps}}
\rfoot{\thepage/\pageref{LastPage}}
\renewcommand{\headrulewidth}{0pt}
\renewcommand{\footrule}{\hrule height 2pt \vspace{2mm}}
\fancyheadoffset[L]{2.25in}
\fancyfootoffset[L]{2.25in}
\lfoot{\today}

%% Include all macros below

\newcommand{\lorem}{{\bf LOREM}}
\newcommand{\ipsum}{{\bf IPSUM}}

%% END MACROS SECTION


\begin{document}
\vspace*{0.2in}

% Title must be 250 characters or less.
\begin{flushleft}
{\Large
\textbf\newline{Integrative QTL analysis of gene expression and chromatin accessibility identifies multi-tissue patterns of genetic regulation} % Please use "sentence case" for title and headings (capitalize only the first word in a title (or heading), the first word in a subtitle (or subheading), and any proper nouns).
}
\newline
% Insert author names, affiliations and corresponding author email (do not include titles, positions, or degrees).
\\
Gregory R. Keele\textsuperscript{1,2,8\Yinyang},
Bryan C. Quach\textsuperscript{1,2,9\Yinyang},
Jennifer W. Israel\textsuperscript{2},
Grace A. Chappell\textsuperscript{7},
Lauren Lewis\textsuperscript{7},
Alexias Safi\textsuperscript{5,6},
Jeremy M. Simon\textsuperscript{2},
Paul Cotney\textsuperscript{2},
Gregory E. Crawford\textsuperscript{5,6},
William Valdar\textsuperscript{2,4,\ddag*},
Ivan Rusyn\textsuperscript{7,\ddag*}
Terrence S. Furey\textsuperscript{2,3,4,\ddag*}
\\
\bigskip
\textbf{1} Curriculum in Bioinformatics and Computational Biology, University of North Carolina at Chapel Hill, Chapel Hill, NC, United States of America
\\
\textbf{2} Department of Genetics, University of North Carolina at Chapel Hill, Chapel Hill, NC, United States of America
\\
\textbf{3} Department of Biology, University of North Carolina at Chapel Hill, Chapel Hill, NC, United States of America
\\
\textbf{4} Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, Chapel Hill, NC, United States of America
\\
\textbf{5} Department of Pediatrics, Duke University, Durham, NC, United States of America
\\
\textbf{6} Center for Genomic and Computational Biology, Duke University, Durham, NC, United States of America
\\
\textbf{7} Department of Veterinary Integrative Biosciences, Texas A\&M University, College Station, TX, United States of America
\\
\textbf{8} The Jackson Laboratory, Bar Harbor, ME, United States of America
\\
\textbf{9} Center for Omics Discovery and Epidemiology, Research Triangle Institute (RTI) International, Research Triangle Park, NC, United States of America
\\
\bigskip

% Insert additional author notes using the symbols described below. Insert symbol callouts after author names as necessary.
% 
% Remove or comment out the author notes below if they aren't used.
%
% Primary Equal Contribution Note
\Yinyang These authors contributed equally to this work.

% Additional Equal Contribution Note
% Also use this double-dagger symbol for special authorship notes, such as senior authorship.
\ddag These authors also contributed equally to this work.

% Current address notes
%\textcurrency Current Address: Dept/Program/Center, Institution Name, City, State, Country % change symbol to "\textcurrency a" if more than one current address note
% \textcurrency b Insert second current address 
% \textcurrency c Insert third current address

% Deceased author note
%\dag Deceased

% Group/Consortium Author Note
%\textpilcrow Membership list can be found in the Acknowledgments section.

% Use the asterisk to denote corresponding authorship and provide email address in note below.
* william.valdar@unc.edu; irusyn@tamu.edu; tsfurey@email.unc.edu

\end{flushleft}
% Please keep the abstract below 300 words
\section*{Abstract}
Gene transcription profiles across tissues are largely defined by the activity of regulatory elements, most of which correspond to regions of accessible chromatin. Regulatory element activity is in turn modulated by genetic variation, resulting in variable transcription rates across individuals. The interplay of these factors, however, is poorly understood. Here we characterize expression and chromatin state dynamics across three tissues---liver, lung, and kidney---in 47 strains of the Collaborative Cross (CC) mouse population, paying particular attention to the regulation of these dynamics by expression quantitative trait loci (eQTL) and chromatin QTL (cQTL). QTL whose allelic effects were common across tissues were detected for 1,101 genes and 133 chromatin regions. Also detected were tissue-selective QTL (potentially tissue-specific), for both genes and chromatin, including a local eQTL for \textit{Pik3c2g} detected in all three tissues but with distinct allelic effects. Leveraging the overlapping measurement of gene expression and chromatin accessibility on the same mice from multiple tissues, we used mediation analysis to identify chromatin and gene expression intermediates of eQTL effects. Based on QTL and mediation analyses over multiple tissues, we propose a causal model for the distal genetic regulation of \textit{Akr1e1}, a gene involved in glycogen metabolism, through a zinc finger protein and chromatin intermediates. This analysis demonstrates the complexity of transcriptional and chromatin dynamics and their regulation over multiple tissues, as well as the value of the CC and related genetic resource populations for identifying specific regulatory mechanisms in biological data.


% Please keep the Author Summary between 150 and 200 words
% Use first person. PLOS ONE authors please skip this step. 
% Author Summary not valid for PLOS ONE submissions.   
\section*{Author summary}
Genetic variation can drive alterations in gene expression levels and chromatin accessibility, the latter of which defines gene regulatory elements genome-wide. The same genetic variants may associate with both molecular events, and these may be connected within the same causal path: a variant that reduces promoter region chromatin accessibility, potentially by affecting transcription factor binding,  will lower expression of that gene. Moreover, these causal regulatory paths can differ between tissues depending on tissue-specific function and cellular activity. We identify cross-tissue and tissue-selective genetic regulators of gene expression and chromatin accessibility in liver, lung, and kidney tissues using a panel of genetically diverse inbred mouse strains. Further, we identify a number of candidate causal mediators of the genetic regulation of gene expression, including a zinc finger protein that helps silence the \textit{Akr1e1} gene. Our analyses are consistent with chromatin accessibility playing a role in the regulation of transcription. Our study demonstrates the power of genetically diverse mouse populations, such as the Collaborative Cross, for large-scale studies of genetic drivers of gene regulation that underlie complex phenotypes, as well as identifying causal intermediates that drive variable activity of specific genes and pathways.

\linenumbers

% Use "Eq" instead of "Equation" for equation citations.
\section*{Introduction}
Determining the mechanisms by which genetic variants drive molecular, cellular, and physiological phenotypes has proved to be challenging \cite{Schadt2009}. These mechanisms can be informed by genome-wide experiments that provide data on variations in molecular and cellular states in genotyped individuals. Most examples of such data, though, are largely observational, due in part to constraints of specific populations (\eg, humans), the limitations of existing experimental technologies, and the challenge of coordinating large numbers of experiments with multiple levels of data \cite{Schaid2018}. 
One approach to shed light on these dynamics is to pair complementary datasets from the same individuals and perform statistical mediation analysis (\eg, \cite{Baron1986, Mackinnon2007}), which has increasingly been used in genomics \cite{Richmond2016}. These analyses can identify putative causal relationships rather than correlational interactions, providing meaningful and actionable targets in terms of downstream applications in areas such as medicine and agriculture.

In human data, co-occurence of QTL across various multi-omic data has been used to assess potentially related and connected biological processes; examples include gene expression with chromatin accessibility \cite{Degner2012} or regulatory elements \cite{Pai2015},  and ribosome occupancy with protein abundances \cite{Battle2015}.
More formal integration through statistical mediation analyses have also been used to investigate relationships between levels of human biological data, such as distal genetical regulation through local gene expression \cite{Battle2014,Yang2017}, and eQTL with regulatory elements \cite{Alasoo2017, Roytman2018, Wu2018}, and physiological phenotypes, such as cardiometabolic traits \cite{Raulerson2019}.

Though genetic association studies of human populations have been highly successful \cite{Visscher2017}, animal models allow for more deliberate control of confounding sources of variation, including experimental conditions and population structure, and thereby provide a potentially powerful basis for detecting associations and even causal linkages.
To aid in this effort, genetically-diverse mouse population resources have been established, including the Collaborative Cross (CC) \cite{Churchill2004,Hall2012,Srivastava2017} and the Diversity Outbred (DO) population \cite{Churchill2012}. The CC and the DO are multiparental populations (MPP), derived from the same eight founder strains (short names in parentheses): A/J (AJ), C57BL/6J (B6), 129S1/SvImJ (129), NOD/ShiLtJ (NOD), NZO/H1LtJ (NZO), CAST/EiJ (CAST), PWK/PhJ (PWK), and WSB/EiJ (WSB). The CC are recombinant inbred strains and therefore replicable across and within studies; the DO are largely heterozygous, outbred animals, bred with a random mating strategy that seeks to maximize diversity. 
MPPs similar to the CC or DO have also been developed in other species, including rats, \textit{Arabidopsis}, \textit{Drosophila}, and yeast, and the use of MPPs in model organism research has accelerated significantly in recent years (\cite{DeKoning2017} and refs therein).

As with humans, it is only recently that studies on MPPs have been the subject of mediation analyses connecting genetic variants with measurements from genomic assays.
%\GKinline{Similar to studies of humans, these} mouse resources have \GKinline{only recently} been the subject of mediation analyses across genomic assays.
A genome-wide mediation approach in 192 DO mice was used to link transcriptional and post-translational regulation of protein levels \cite{Chick2016}; the CC were used to confirm results by showing that estimates of founder haplotype effects from each of the related populations corresponded. More recently, mediation analysis was used to connect chromatin accessibility with gene expression in embryonic stem cells derived from DO mice \cite{Skelly2019}. 
The CC and DO, and MPPs more broadly, have well-characterized haplotype structures that provide a unique opportunity for studying mediation at the haplotype level. This is potentially advantageous because haplotypes can capture genetic variation and its effects more comprehensively than can individual SNPs \cite{Zhang2014}, the latter being the predominant basis for comparable mediation analyses in humans.
%\TF{}{Do you mean that these allow for a more direct tracing back to the parental strains? Arguably, though, you can lose resolution as the haplotype blocks are larger than in human.}
%\GK{}{Reviewer 1 mentioned providing a review of MPP in the context of mediation analysis ("significance of this concept in MPP"). Which is hard because we already cite pretty much what there is with the DO papers from Churchill. This was my attempt to describe added benefit that MPP could provide to mediation. I agree that the resolution will be worse, particularly in the CC, but our ability to fit a more complex genetic model (haplotypes) is stably allows for essentially better pattern matching between the levels of data. Whereas in humans, it would be a more complex process of looking for similar SNP effects and the LD patterns. I think all that gets nicely folded up into a haplotype-based mediation analysis.}

Here we use a sample composed of a single male mouse from 47 CC strains to investigate tissue-specific dynamics between gene expression and chromatin accessibility, as determined by Assay for Transposase Accessible Chromatin sequencing (ATAC-seq), in lung, liver, and kidney tissues. We detect QTL underlying gene expression and chromatin accessibility variation across the strains and assess support for mediation of the effect of eQTL through chromatin accessibility using a novel implementation of previous methods used in the DO \cite{Chick2016}. Additionally, we detect gene mediators of distal-eQTL. 
%We identify and characterize examples of strong mediation, as well as co-localizing but independent eQTL and cQTL. 
These findings demonstrate the experimental power of the CC resource for integrative analysis of multi-omic data to determine genetically-driven phenotype variation, despite limited sample size, and provide support for continued use of the CC in larger experiments going forward.

% Results and Discussion can be combined.
\section*{Results}
\subsection*{Differential gene expression and chromatin accessibility}

\subsubsection*{Gene expression and chromatin accessibility cluster by tissue}
Gene expression and chromatin accessibility were measured with RNA-seq and ATAC-seq assays, respectively, from whole lung, liver, and kidney tissues in one male mouse from each of 47 CC strains (Fig \ref{fig:overview}). 
The use of only male mice was due to practical constraints; results for females may differ \cite{Chick2016}. As the function of each tissue is quite distinct, we expected that these data would reflect tissue-specific differences. Principal components analysis (PCA) of each of the gene expression and chromatin accessibility profiles showed that the samples clearly clustered by tissue (\nameref{S_pca}). 

\begin{figure}[!h]
\caption{{\bf Diagram of the experiment and analyses.}
RNA-seq and ATAC-seq were performed using liver, lung, and kidney tissues of males from 47 CC strains. Each CC strain was derived from an inbreeding funnel, and thus represents a recombinant inbred mosaic of the initial eight founder haplotypes. Differential analyses followed by pathway enrichment analyses were performed to identify biological pathways enriched in differentially expressed genes and accessible chromatin regions. QTL and mediation analyses were performed to identify regions that causally regulate gene expression and chromatin accessibility.}
\label{fig:overview}
\end{figure}

\subsubsection*{Differentially expressed genes strongly correspond with accessible chromatin regions} 
Differential expression (DE) and differentially accessible region (DAR) analysis were performed between the three tissues (\nameref{S_de}), and revealed between 3,564 - 5,709 DE genes and 28,048 - 40,797 DARs (FWER $\le 0.1$). For both expression and chromatin accessibility, liver and kidney tissues were the most similar, while lung and liver were the most distinct, also seen in the PCA plots (\nameref{S_pca}). Pathway analyses showed many between-tissue differences related to metabolic and immune-related pathways (FWER $\le 0.1$), reflecting the distinct demands of each tissue. Energy metabolism pathways were more active in liver and kidney and immune-related pathways were more pronounced in lung, as expected. We compared the concordance between DE genes and DARs genome-wide and observed that most DE gene promoters do not show significant differences in chromatin accessibility (\nameref{S_de_concordance}). In cases with significant variability in accessibility at the promoter of a DE gene, though, the vast majority agree in direction (\ie, higher expression with greater accessibility).

\subsection*{QTL detection}

\subsubsection*{Gene expression}
The impact of genetic variation on gene expression was evaluated by eQTL mapping. This was done at three levels of stringency and emphasis: 1) at the level of the local region of a gene, defined as within 10Mb of the gene transcription start site (TSS), and hereafter termed Analysis L; 2) at the level of the chromosome on which the gene is located (Analysis C); and 3) at level of the genome (Analysis G) (details in \textbf{Methods}).
After filtering out lowly expressed genes, the number of genes examined in eQTL mapping was 8401 for liver, 11357 for lung, and 10092 for kidney (UpSet plot \cite{Conway2017} in \nameref{S_upset_outcomes}A).

Analysis L detected local-eQTL for 19.8\% of genes tested in liver, 16.6\% in lung, and 20.8\% in kidney (\nameref{S_eqtl}).
Most genes with local-eQTL were observed in only one tissue (\nameref{S_upset_qtl}A).
Analysis C, which was more stringent, additionally detected intra-chromosomal distal-eQTL, while Analysis G, the most stringent, additionally detected inter-chromosomal distal-eQTL (\nameref{S_eqtl}).
Genomic locations of eQTL detected for each tissue, excluding the intra-chromosomal distal-eQTL detected by Analysis C, are shown in Fig \ref{fig:qtl_results}A[top]. See \nameref{S_grid_plot_lenient} and \nameref{S_eqtl_lenient} for eQTL counts with FDR $\le 0.2$.

%\WV{We also identified eQTL with Analyses C and G using a more relaxed significance threshold (FDR $<$ 0.2; \nameref{S_grid_plot_lenient}A and \nameref{S_eqtl_lenient}), which increased the number of distal-eQTL detected by a greater extent than local-eQTL. 
%Using Analysis L and requiring genome-wide significance, the percentage of tested genes with local-eQTL were 8.4\% for liver, 6.3\% for lung, and 9.5\% for kidney (\nameref{S_eqtl}). 
%These percentages increase to 19.8\%, 16.6\%, and 20.8\% respectively with the less stringent Analysis L at chromosome-wide significance. 
%Additionally, we classify distal-eQTL located on the same chromosome as the associated gene as intra-chromosomal, and otherwise inter-chromosomal.
%}{Do we need any of this?}

%\begin{figure}[h!]
%\caption{\textbf{Detected QTL are largely local for both gene expression (A) and chromatin accessibility (B).} 
%QTL detected through all analyses, excluding intra-chromosomal distal-QTL detected through Analysis C, are included. The y-axis represents the genomic position of the gene or chromatin site, and the x-axis represents the genomic position of the QTL. Local-QTL appear as dots along the diagonal and distal-QTL as off-diagonal dots.
%\label{fig:grid_plot}}
%\end{figure}

\begin{figure}[h!]
\caption{\textbf{QTL mapping results and comparison of matched pairs from lung and kidney.}
%% A 
(A) Detected QTL are largely local for both gene expression and chromatin accessibility. QTL detected through all analyses, excluding intra-chromosomal distal-QTL detected through Analysis C, are included. The y-axis represents the genomic position of the gene or chromatin site, and the x-axis represents the genomic position of the QTL. Local-QTL appear as dots along the diagonal and distal-QTL as off-diagonal dots. The gray dashed box highlights lung and kidney QTL results, which are further explored in B-F.
%% B
(B) Highly significant QTL map nearby the gene TSSs and chromatin window midpoints. Results from Analysis G are shown here. The red dashed line represents $\pm$ 10Mb from the trait genomic coordinate used to classify a QTL as local or distal. See \nameref{S_dist} for all tissues as well as results from Analysis C.
%% C
(C) Local QTL have larger effect sizes than distal. Only results from Analysis G are included here. See \nameref{S_effect_size_status} for all tissues and results from Analysis C.
%% D
(D) Consistent genetic regulation of gene expression and chromatin accessibility was observed between lung and kidney tissue, based on an enrichment in positively correlated founder haplotype effects for QTL paired across the two tissues. QTL from all the Analyses were considered here. See \nameref{S_qtl_pair_histograms} for all tissue pairings.
%% E
(E) QTL observed in both lung and kidney that are highly correlated tend to map close to each other, consistent with representing genetic variation active in both tissues. See \nameref{S_qtl_cor_by_distance_comparison} for all tissue pairings.
%% F
(F) The effect sizes of QTL paired from lung and kidney are significantly correlated. See \nameref{S_qtl_effect_size_comparison} for all tissue comparisons.}
\label{fig:qtl_results}
\end{figure}

\subsubsection*{Chromatin accessibility}
To determine genetic effects on chromatin structure, genomic regions were divided into $\sim$300 base pair windows and analyses similar to eQTL were used to detect chromatin accessibility QTL (cQTL). The number of chromatin regions tested were 11448 in liver, 24426 in lung, and 17918 in kidney. The overlap in chromatin windows tested across tissues is described in \nameref{S_upset_outcomes}B. 
The differences in genes and chromatin regions tested within each tissue likely reflects both biological and technical factors that distinguish the tissue samples.
Overall, there were substantially fewer cQTL detected compared with eQTL for all tissues (Fig \ref{fig:qtl_results}A[bottom]; \nameref{S_cqtl}, \nameref{S_cqtl_lenient}). As with eQTL, cQTL were more likely to be local than distal (66 - 94.1\% local-cQTL for Analysis G; 75 - 90\% local-cQTL for Analysis C).

\subsubsection*{Local-QTL have stronger effects than distal-QTL}
For QTL detected on the same chromosome as the gene or chromatin region (intra-chromosomal), the strongest associations were observed within 10Mb of the gene TSS or chromatin window midpoint (Fig \ref{fig:qtl_results}B and \nameref{S_dist}). Intra-chromosomal distal-QTL had reduced statistical significance, more consistent with inter-chromosomal distal-QTL. This dynamic between local- and distal-QTL is also observed when using the QTL effect size as a summary of strength, which is likely biased upward due to the Beavis effect in 47 strains \cite{Keele2019}, shown in Fig \ref{fig:qtl_results}C and \nameref{S_effect_size_status}.

\subsubsection*{QTL driven by extreme effects of CAST and PWK}
For all QTL we estimated the effects of the underlying founder haplotypes. Consistent with previous studies (\eg, \cite{Aylor2011}), the CAST and PWK haplotypes had higher magnitude effects compared with the classical inbred strains. This pattern was observed for both local- and distal-eQTL (\nameref{S_qtl_effects_abs}A) and local-cQTL; the numbers of detected distal-cQTL were too low to produce clear trends (\nameref{S_qtl_effects_abs}B).

\subsection*{QTL paired across tissues and correlated haplotype effects}

For a given trait, QTL from different tissues were paired based on co-localizing to approximately the same genomic region. For local-QTL, both had to be within the local window, defined as 10Mb around the gene TSS or chromatin region midpoint, resulting in a maximum distance of 20Mb between QTL. For distal-eQTL, both had to be within 10Mb of each other. 
To assess the consistency between the QTL paired across tissue, the correlations between the founder haplotype effects were tested (FDR $\le 0.1$; Fig \ref{fig:qtl_results}D and \nameref{S_qtl_pair_histograms}), revealing significant positive correlations as would be expected for QTL acting similary in both tissues (multi-tissue QTL).
For local-eQTL, we found significant haplotype effect correlations between 345 of 761 possible pairs (45.3\%) in liver/lung, 623 of 1206 (51.7\%) in liver/kidney, and 497 of 1025 (48.5\%) in lung/kidney. For distal-eQTL, we found significant correlations between 21 of 61 possible pairs (31.8\%) in liver/lung, 34 of 120 (28.8\%) in liver/kidney, and 16 of 59 (27.1 \%) in lung/kidney. For cQTL, the vast majority of correlated pairs were local, with 47 of 55 (85.5\%) possible pairs in liver/lung, 48 of 56 (85.7\%) in liver/kidney, and 118 of 142 (83.1\%) in lung/kidney being significantly correlated. Only 4 distal-cQTL pairs were observed, all between lung and kidney, with three of the four pairs possessing nominal correlation $p$-values $\le$ 0.05. The effect sizes of paired QTL varied across tissues, though for any given tissue pairing, the effect sizes were significantly correlated (least significant $p$-value =\num{4.6e-8}; Fig \ref{fig:qtl_results}F and \nameref{S_qtl_effect_size_comparison}). No significantly negatively correlated QTL pairs were detected after accounting for multiple testing (FDR $\le 0.1$).
%, consistent with being tissue-specific QTL with transgressive effects when compared across tissues.}

%\subsubsection*{Multi-tissue QTL pairs share similar haplotype effects across tissues}

 %(FDR $\le 0.1$) for 345 (45.3\%; liver/lung), 623 (51.7\%; liver/kidney), and 497 (48.5\%; lung/kidney) local-eQTL pairs and 21 (31.8\%; liver/lung), 34 (28.8\%; liver/kidney), and 16 (27.1\%; lung/kidney) distal-eQTL pairs. Highly correlated QTL pairs were very close in genomic distance to each other (Fig \ref{fig:qtl_results}E and \nameref{S_qtl_cor_by_distance_comparison}), suggesting that the underlying causal variants are the same or linked. For local-cQTL, 47 (85.5\%; liver/lung), 48 (85.7\%; liver/kidney), and 118 (83.1\%; lung/kidney) significantly positively correlated pairs were identified. The correlations between distal-cQTLs were not formally tested because there were only four pairs; however, three of the four pairs had correlations greater than 0.5. No significantly negatively correlated QTL pairs were detected, after accounting for multiple testing (FDR $\le 0.1$)\GKinline{, consistent with being tissue-specific QTL with transgressive effects when compared across tissues}.

\subsubsection*{\textit{Cox7c}: Consistent haplotype effects for multi-tissue local-eQTL}
Cytochrome c oxidase subunit 7C (\textit{Cox7c}) is an example of a gene that possessed local-eQTL with highly correlated effects in all three tissues (Fig \ref{fig:correlated_local_eqtl}A).
The local-eQTL consistently drove higher expression when the CAST haplotype was present, intermediate expression with 129, NOD, and NZO haplotypes, and lower expression with A/J, B6, PWK, and WSB haplotypes. Though the haplotype effects on relative expression levels within each tissue were consistent, we noted that the expression level was significantly higher in liver compared with both lung ($q = \num{5.41e-18}$) and kidney ($q = \num{6.25e-19}$). Ubiquitin C (\textit{Ubc}) is another example of a gene with consistent local-eQTL detected in all three tissues (\nameref{S_ubc_correlated_eqtl}).

\begin{figure}[h!]
\caption{\textbf{Examples of genes with local-eQTL observed in all three tissues.} 
(A) \textit{Cox7c} possesses local-eQTL with highly correlated haplotype effects across all three tissues, supportive of shared causal origin. (B) \textit{Slc44a3} has a more complicated pattern of local-eQTL founder effects across the tissues, with correlated effects shared between lung and kidney, and transgressive effects in the liver eQTL by comparison, consistent with distinct casual variants comparing liver to lung and kidney. For both genes, the expression data are plotted with bars representing the interquartile ranges of likely founder haplotype pair (diplotype). Differential expression between tissues is highlighted. The haplotype association for each tissue is also included near the gene TSS with variant association overlaid. The most statistically rigorous method that detected the QTL (Analysis L, C, or G) is also included. The red tick represents the gene TSS, the black tick represents the variant association peak, and the colored tick represents the haplotype association peak. Founder effects estimated as constrained best linear unbiased predictions (BLUPs) are also included with their pairwise correlations. Estimated effects were generally consistent with the expression data.
\label{fig:correlated_local_eqtl}}
\end{figure}

\subsubsection*{\textit{Slc44a3} and \textit{Pik3c2g}: Tissue-specific local-eQTL}
We refer to QTL detected in only a subset of the three tissues as tissue-selective because we cannot definitively show that the QTL is not present in the other tissues, but undetected.
Uncorrelated QTL pairs provide stronger support of tissue-specificity.
For example, the solute carrier family 44, member 3 (\textit{Slc44a3}) gene has local-eQTL effects that are correlated in lung and kidney but anticorrelated liver (Fig \ref{fig:correlated_local_eqtl}B), suggesting the liver eQTL could be transgressive \cite{Rieseberg1999} relative to the eQTL in lung and kidney, whereby the effects of the founder haplotype are reversed. 
For \textit{Slc44a3}, CAST, PWK, and WSB haplotypes result in higher expression in liver, but lower expression in lung and kidney.
The local-eQTL for \textit{Slc44a3} were more similar in location in lung and kidney, whereas the liver eQTL was more distal to the gene TSS. Overall, the expression data, estimated effects, and patterns of association are consistent with lung and kidney sharing a causal local-eQTL that is distinct from the one in liver. 

The CC founder strains all possess contributions from three subspecies of \textit{M. musculus}: \textit{domesticus} (\textit{dom}), \textit{castaneus}, (\textit{cast}), and \textit{musculus} (\textit{mus}) \cite{Yang2011}.  Allele-specific gene expression in mice descended from the CC founders often follow patterns that matched the subspecies inheritance at the gene regions \cite{Wang2012,Crowley2015}.
We found that phosphatidylinositol-4-phosphate 3-kinase catalytic subunit type 2 gamma (\textit{Pik3c2g}), a gene of interest for diabetes-related traits \cite{Braccini2015}, had tissue-specific local-eQTL in all three tissues that closely matched the subspecies inheritance at their specific genomic coordinates. The local-eQTL in all three tissues were all pair-wise uncorrelated (Fig \ref{fig:pik3c2g}). Further, expression of \textit{Pik3c2g} varied at statistically significant levels for liver versus lung ($q = \num{3.12e-13}$) and lung versus kidney ($q = \num{3.55e-6}$). 
In lung, the CAST haplotype (\textit{cast} subspecies lineage) resulted in higher expression, consistent with a \textit{cast} versus \textit{dom}/\textit{mus} allelic series. In kidney, the B6, 129, NZO, and WSB haplotypes  (\textit{dom} subspecies) resulted in higher expression, whereas A/J and NOD (\textit{mus}), CAST (\textit{cast}), and PWK (\textit{dom}) haplotypes showed almost no expression, mostly consistent with a \textit{dom} versus \textit{cast}/\textit{mus} allelic series. The PWK founder appears inconsistent, but we note that the QTL peak was in a small \textit{dom} haplotype block interspersed within a broader \textit{mus} region. The expression level in kidney of \textit{Pik3c2g} in PWK was low, similar to A/J and NOD, suggesting that the causal variant is located in the nearby region, where all three have \textit{mus} inheritance and, notably, where the peak variant association occurred. 
In liver, the B6 haplotype resulted in lower expression compared with the other haplotypes. Interestingly, the liver eQTL was in a region that contained a recombination event between \textit{dom} and \textit{mus}, only present in B6, which may explain the unique expression pattern.

\begin{figure}[h!]
\caption{\textbf{\textit{Pik3c2g} possesses tissue-specific local-eQTL.} Local-eQTL for \textit{Pik3c2g} were detected in all three tissues in the 3Mb region surrounding its TSS. The genomes of the CC founders can be simplified in terms of contributions from three subspecies lineages of \textit{M. musculus}: \textit{dom} (blue), \textit{cast} (yellow), and \textit{mus} (red). The effects of each local-eQTL matched the subspecies contributions near the eQTL coordinates, with low expressing subspecies alleles colored teal and high expressing alleles colored salmon, consistent with local-eQTL for \textit{Pik3c2g} being distinct and tissue-specific. The gene expression data are represented as interquartile range bars, categorized based on most likely diplotype at the eQTL for each CC strain. Haplotype and variants associations are included for each tissue, with the red tick representing the \textit{Pik3c2g} TSS, black ticks representing variant association peak, and colored ticks representing the haplotype association peak. The most rigorous procedure to detect each QTL (Analysis L, C, or G) is reported. Founder effects, estimated as constrained BLUPs, were consistent with the expression data, and uncorrelated across the tissues.
\label{fig:pik3c2g}}
\end{figure}

\subsubsection*{\textit{Akr1e1} and \textit{Per2}: Consistent haplotype effects for multi-tissue distal-eQTL}
Founder haplotype effects that correlate across tissues can provide stronger confirmation of distal-QTL, even those with marginal significance in any single tissue. 
For example, the Aldo-keto reductase family 1, member E1 (\textit{Akr1e1}; Fig \ref{fig:correlated_distal_eqtl}A) gene is located on chromosome 13 with no local-eQTL detected in any tissue. Distal-eQTL in all tissues were detected for \textit{Akr1e1} that localized to the same region on chromosome 4. The founder haplotype effects of the distal-eQTL were all highly correlated, with A/J, B6, 129, and CAST haplotypes corresponding to higher \textit{Akr1e1} expression. Across tissues, expression was significantly higher in liver and kidney compared with lung ($q = \num{1.63e-19}$ and $q = \num{4.40e-34}$, respectively).
Another example is the Period circadian clock 2 (\textit{Per2}; Fig \ref{fig:correlated_distal_eqtl}B) gene, which possessed intra-chromosomal distal-eQTL approximately 100Mb away from the TSS that were detected in all three tissues at a lenient chromosome-wide significance (Analysis C; FDR $\le 0.2$). The founder haplotype effects were significantly correlated among the tissues, characterized by high expression with the NOD and PWK haplotypes present. Together, these findings provide strong validation for the distal-eQTL, which would commonly not be detected based on tissue-stratified analyses.
Other unique patterns of distal genetic regulation were detected, such as for Ring finger protein 13 gene (\textit{Rnf13}) with tissue-selective distal-eQTL, described in \nameref{S_rnf13_distal_eqtl}.

\begin{figure}[h!]
\caption{\textbf{Examples of genes with distal-eQTL effect patterns across tissues.} 
(A) \textit{Akr1e1} has highly significant distal-eQTL detected on chromosome 4 in all three tissues with correlated founder effects. (B) \textit{Per2} has intra-chromosomal distal-eQTL leniently detected 100Mb away from the TSS, also with highly correlated founder effects across the tissues, providing further support that the distal-eQTL are not false positives. 
%\textit{Rnf13} has a liver-specific distal-eQTL on chromosome 13 detected after conditioning on its local-eQTL on chromosome 2, and a lung-specific distal-eQTL on chromosome X, each with distinct founder effect patterns (C). 
Expression data are represented as interquartile ranges for most likely diplotype, with differential expression noted when significant. Haplotype associations for each tissue distal-eQTL combination are shown, with the most rigorous statistical procedure for detection reported (Analysis L, C, or G). Red ticks signify the gene TSS and black ticks represent that eQTL peak. Fit founder effects, estimated as BLUPs, are included, along with the pairwise correlations of the eQTL. 
%The black asterisk marks the high B6 effect that drives the distal-eQTL in liver after conditioning on the local-eQTL for \textit{Rnf13}.
\label{fig:correlated_distal_eqtl}}
\end{figure}

\subsection*{Mediation of eQTL by chromatin}

An advantage of measuring gene expression and chromatin accessibility in the same mice and tissues is the subsequent ability to examine the relationships between genotype, chromatin accessibility, and gene expression using integrative methods such as mediation analysis.
We considered two possible models for the mediation of eQTL effects and assessed the evidence for each in our data.
In the first model, proximal chromatin state acts as a mediator of local-eQTL (Fig \ref{fig:graph}A). That is, the local-eQTL for a gene affects that gene's expression by, at least in part, altering local chromatin accessibility. To test for this, we used an approach adapted from studies in the DO \cite{Chick2016} and applied it to local-eQTL detected through Analysis L
%\WV{}{I still find the mediation arrow liable to mislead; mediation is the combination of blue and red. It wouldn't hurt to label the arrows.} 
(see \nameref{S3_Appendix} for greater detail). 

\begin{figure}[h!]
\caption{{\bf Simple mediation models for the genetic regulation of gene expression.} 
(A) Mediation of the local-eQTL through chromatin accessibility in the region of the gene is consistent with genetic variation influencing the accessibility of gene $j$ to the transcriptional machinery. (B) Mediation of distal-eQTL through the transcription of genes local to the QTL could be explained by transcription factor activity.
\label{fig:graph}}
\end{figure}

\subsubsection*{\textit{Ccdc137} and \textit{Hdhd3}: Local-eQTL driven by local chromatin accessibility}
Across the three tissues, between 13-42 local-eQTL showed evidence of mediation through proximal accessible chromatin regions at genome-wide significance, and 35-106 at chromosome-wide significance (\nameref{S_mediation}).
The coiled-coil domain containing 137 gene (\textit{Ccdc137}) is a strong example of this type of mediation. Fig \ref{fig:ccdc137_mediation} shows the genome scans that identify both the local-eQTL for \textit{Ccdc137} and the cQTL near it. 
The significance of the eQTL was sharply reduced when we conditioned on chromatin accessibility, and the significance of the cQTL was stronger than the eQTL, as expected by the proposed mediator model.

\begin{figure}[h!]
\caption{\textbf{\textit{Ccdc137} local-eQTL is mediated by proximal chromatin accessibility.} \textit{Ccdc137} expression and the chromatin accessibility in the proximal region were highly correlated ($r = 0.71$). Genome scans for \textit{Ccdc137} expression (yellow), nearby chromatin accessibility (blue), and chromatin mediation of the \textit{Ccdc137} local-eQTL (red) in lung tissue. The local-eQTL and local-cQTL for the chromatin region at the TSS of \textit{Ccdc137} (red tick) are over-lapping, and have highly correlated haplotype effects ($r = 0.98$). The steep drop in the statistical association with expression, represented as logP, at the chromatin site in the mediation scan supports chromatin mediation of \textit{Ccdc137} expression, depicted as a simple graph [top right]. The QTL and mediation signals were detected at genome-wide significance. \label{fig:ccdc137_mediation}}
\end{figure}

Establishing mediation requires more than mere co-localization of eQTL and cQTL.
For example, local-eQTL and co-localizing cQTL were identified for both haloacid dehalogenase-like hydrolase domain containing 3 (\textit{Hdhd3}) in liver (\nameref{S_colocalization}A) and acyl-Coenzyme A binding domain containing 4 (\textit{Acbd4})  in kidney (\nameref{S_colocalization}B). Comparing the statistical associations at the loci for eQTL and cQTL, and the corresponding haplotype effects, however, showed better correspondence in the case of \textit{Hdhd3} than for \textit{Acbd4}: a strong mediation signal was detected for \textit{Hdhd3}, indicated by the decreased eQTL association when conditioning on the chromatin state, whereas mediation was not detected for \textit{Abcd4}, consistent with expression and chromatin accessibility having causally different origins.

\subsection*{Mediation of eQTL by expression} 

Mediation was also tested for distal-eQTL, detected through Analysis G, by evaluating the expression of nearby genes as candidate mediators \cite{Keller2018}, as shown in Fig \ref{fig:graph}B. 

\subsubsection*{Zinc finger protein intermediates detected for \textit{Ccnyl1}}
Eight genes were identified with mediated distal-eQTL (\nameref{S_exmediation}). For example, the distal-eQTL of cyclin Y-like 1 (\textit{Ccnyl1}) was mediated by the expression of zinc finger protein 979 (\textit{Zfp979}), a putative transcription factor (\nameref{S_ccnyl1_exmediation}). The founder haplotype effects at the distal-eQTL of \textit{Ccnyl1} were highly correlated with the local-eQTL effects for \textit{Zfp979}, though with a reduction in overall strength, in accordance with the proposed mediation model.

\subsection*{Mediation of eQTL by both expression and chromatin}
The QTL and mediation results for all three tissues are depicted as circos plots \cite{Gu2014} in Fig \ref{fig:circos_plot}. The local-QTL and chromatin mediators were distributed across the genome unevenly, aggregating in pockets. In particular, a high concentration of cQTL and chromatin mediation were detected in all three tissues along chromosome 17, which corresponds to the immune-related major histocompatibility (MHC) region in mouse. However, most of the chromatin-mediated genes in this region are not histocompatibility genes (Files S28 - S30). Patterns of distal-QTL and gene mediation have tissue-selective patterns, though consistent regions are observed across the tissues, such as the \textit{Idd9} region on chromosome 4 described in \cite{HamiltonWilliams2010}, which contains multiple zinc finger proteins (ZFPs) and regulates genes such as \textit{Ccnyl1} and \textit{Akr1e1}, described in greater detail below.

\begin{figure}[h!]
\caption{\textbf{Summaries of QTL and mediation analyses.} Circos plots of eQTL (yellow), cQTL (blue), and mediation (red) in lung, liver, and kidney. The two outer rings of dots represent local-eQTL and local-cQTL detected by Analysis L at chromosome-wide significance, with red lines between connecting genes and chromatin sites for which chromatin mediation was detected. The inner circle contains connections representing distal-eQTL, distal-cQTL, and gene-gene mediation from Analysis G. Thick lines represent QTL and mediators with permutation-based $p$-value (permP) $< \num{1e-5}$. The detected signals were primarily local, which also tended to be stronger than the observed distal signals. Fewer QTL and mediators were detected in liver tissue. 
Genes highlighted within the text, such as \textit{Akr1e1}, are indicated at their genomic coordinates.
\label{fig:circos_plot}}
\end{figure}

\subsubsection*{Genetic regulation of \textit{Akr1e1} expression includes zinc finger protein and chromatin intermediates}
As detailed above, \textit{Akr1e1} possessed a strong distal-eQTL in all three tissues located on chromosome 4 in the region of 142.5 - 148.6Mb with significantly correlated haplotype effects. Mediation analysis suggested this effect is mediated in lung through activity of Zinc finger protein 985 (ZFP985), whose gene is also located on chromosome 4 at 147.6Mb (\nameref{S_akr1e1_exmediation}). 

ZFP985 possesses an N-terminal Kr{\"u}ppel-associated box (KRAB) domain, representing a well-characterized class of transcriptional regulators in vertebrates \cite{Ecco2017}, that recruit histone deacetylases and methyltransferases, inducing a heterochromatic state associated with regulatory silencing. 
\textit{Akr1e1} and \textit{Zfp985} expression were negatively correlated ($r = -0.69$), consistent with ZFP985 inhibiting \textit{Akr1e1} expression. This same distal-eQTL and mediator relationship for \textit{Akr1e1} was observed in kidney tissue from 193 DO mice (\nameref{S_do_akr1e1}).
It was previously postulated that \textit{Akr1e1} is distally regulated by the Reduced expression 2 gene (\textit{Rex2}), also a ZFP that contains a KRAB domain and resides in the same region of chromosome 4 \cite{HamiltonWilliams2013}.
Our distal-eQTL overlap their \textit{Idd9.2} regulatory region, defined as spanning 145.5 - 148.57Mb, mapped with NOD mice congenic with C57BL/10 (B10) \cite{HamiltonWilliams2010}.
The B10 haplotype was found to be protective against development of diabetes, characteristic of NOD mice.
AKR1e1 is involved in glycogen metabolism and the \textit{Idd9} region harbors immune-related genes, suggesting genes regulated by elements in the \textit{Idd9} region may be diabetes-related. 
Consistent with these studies, CC strains with NOD inheritance at the distal-eQTL had low expression of \textit{Akr1e1}, observed in all tissues.
Genetic variation from B10 is not present in the CC, but the closely related B6 founder had high expression of \textit{Akr1e1} like B10.
NZO, PWK, and WSB haplotypes resulted in low \textit{Akr1e1} expression, similar to NOD, while A/J, 129, and CAST haplotypes joined B6 as driving high expression (Fig \ref{fig:akr1e1_full_model}). Lowly expressed \textit{Zfp985}, resulting in low levels of ZFP985, is a strong candidate for driving these effects on \textit{Akr1e1}.

\begin{figure}[h!]
\caption{\textbf{Mediation model for \textit{Akr1e1} distal-eQTL.}  The genetic regulation of \textit{Akr1e1} expression is reconstructed based on relationships observed across the three tissues. Distal-eQTL were detected in all tissues at similar levels of significance. A local-eQTL for \textit{Zfp985} that is proximal to the \textit{Akr1e1} distal-eQTL was observed in lung, and \textit{Zfp985} expression was detected as an anti-correlated mediator of the distal-eQTL, consistent with ZFP985 suppressing \textit{Akr1e1} expression. The chromatin site proximal to the \textit{Akr1e1} TSS has a distal-cQTL detected in kidney. Chromatin accessibility at the site was found to be a significant mediator of \textit{Akr1e1} expression. Combining associations across tissues supports a biological model whereby ZFP985, whose gene is expressed in mice with NOD, NZO, PWK, and WSB haplotypes, silences \textit{Akr1e1} through KRAB domain-induced chromatin remodeling. QTL and mediation genome scans are included, along with sequence phenotypes as interquartile ranges categorized according to most likely diplotype, and modeled founder effects fit as BLUPs. The relative magnitudes of the QTL effect sizes and mediation scores are consistent with the proposed model, with \textit{Zfp985} local-eQTL $>$ distal-cQTL $>$ \textit{Akr1e1} distal-eQTL, and chromatin mediation $>$ mediation through \textit{Zfp985} expression.
\label{fig:akr1e1_full_model}}
\end{figure}
  
Additional sources of genetic regulation of \textit{Akr1e1} were observed for kidney, where a strong distal-cQTL for an accessible chromatin site corresponding to the promoter of \textit{Akr1e1} was detected in the \textit{Idd9.2} region on chromosome 4. Mediation analysis strongly supported chromatin accessibility in this region mediating the distal-eQTL ($\permpmed = \num{2.18e-13}$). The founder effects for the distal-cQTL were highly correlated with the distal-eQTL effects ($r = 0.92$). The relative magnitudes of the QTL effect sizes and mediation $p$-values (\nameref{S_akr1e1_relationships}) support a causal model whereby increased \textit{Zfp985} expression reduces expression of \textit{Akr1e1} by altering chromatin accessibility near the \textit{Akr1e1} promoter (Fig \ref{fig:akr1e1_full_model}). 

\section*{Discussion}

In this study we performed QTL and mediation analyses of gene expression and chromatin accessibility data in liver, lung, and kidney tissue samples from 47 strains of the CC. We examined correlations between founder haplotype effects of overlapping QTL to identify QTL that are likely functionally active in multiple tissues as well as potentially distinct tissue-specific QTL, as is the case with the \textit{Pik3c2g} gene. We detected extensive evidence of chromatin mediation of local-eQTL as well as gene expression mediators underlying distal-eQTL. One unique example is the elucidation of the genetic regulation of \textit{Akr1e1} expression, a gene that plays a role in glycogen metabolism, involving distal inhibition by a zinc finger protein mediator that reduces chromatin accessibility at the TSS. These findings highlight the ability of integrative QTL approaches such as mediation analysis to identify interesting biological findings, including the ability to identify functional candidates for further downstream analysis.

\subsection*{\GKinline{Fewer cQTL detected compared to eQTL}}

\GKinline{As we show, as have others, the effect sizes of cQTL are on average lower than eQTL (see \nameref{S_qtl_effect_sizes_by_method}). The reduced number of cQTL compared to eQTL is likely due to both technical and biological reasons. Technically, the RNA-seq assay measures a distinct class of molecules (mRNAs) that can be accurately extracted from cells, and the resulting sequence reads are then mapped to the transcriptome, which encompasses $<$ 5\% of the genome. In contrast, the transposon incorporation event central to ATAC-seq is enriched in, but not solely limited to, accessible chromatin. Unlike the transcriptome, accessible chromatin can occur anywhere in the genome and regions are defined empirically by the data. Thus, the signal from this assay is more variable and noisy across samples, which impedes our ability to detect cQTL. Biologically, if a variant affects the activity of a regulatory element that alters expression levels, you will invariably detect this as increased/decreased mRNAs in the sample. In contrast, even if a variant affects the accessibility of chromatin, it may do so in a manner that is difficult to detect above the background noise of the assay. In recent studies in the adult brain on the same individuals, 2,154,331 \textit{cis}-eQTL were found for 467 individuals \cite{Richard2012}, whereas only 6,200 cQTL were found, albeit for only 272 individuals \cite{Bryois2018}. While the reduced number of individuals undoubtedly contributed to this lower number of cQTL, it is likely that significantly fewer cQTL would be found in the same number of individuals.}

\subsection*{\GKinline{Reduced cQTL and mediation signal in liver compared to lung or kidney}}

\GKinline{Detection of QTL and mediators is dependent on not only sample size, but also biological and technical factors that are difficult to quantify across the tissues. Potential differences in technical factors limit the ability to make strong biological conclusions based on differences in numbers of QTL or mediators, or the lack thereof, in any given tissue relative to the other tissues. Although it is possible that liver has fewer actual cQTL (and thus fewer chromatin mediators) than lung and kidney, it is also possible that the ATAC-seq data happen to be noisier for this experiment. If there are true biological differences in the number of cQTL and mediator usage among the tissues, they likely reflect tissue-specific multi-level regulatory programs, a complex subject requiring more targeted experiments than used here.}

\subsection*{Joint QTL analysis in multiple tissues}

Multi-tissue QTL analyses are increasingly used in both humans, such as within the GTEx project (\eg,\cite{GTEX2017, Fagny2017}), 
and in mice (\eg, \cite{Huang2009, Sul2013}).
We believe our use of formal statistical tests of the correlation coefficient between the founder haplotype effects of overlapping QTL is novel in defining co-localizing QTL across tissues. This method allows us to identify loci likely representing tissue-specific QTL with unique haplotype effects patterns, as demonstrated for the gene \textit{Pik3c2g} (Fig \ref{fig:pik3c2g}).
This approach can also detect consistent haplotype effects, as with the gene \textit{Per2} (Fig \ref{fig:correlated_distal_eqtl}B), which possessed only marginally significant distal-eQTL in the three tissues, suggesting jointly mapping QTL across all tissues increases power. Formal joint analysis approaches have been proposed, largely implemented for detecting SNP associations, including meta-analysis on summary statistics (\eg, \cite{Fu2012a, Sul2013}) and fully joint analysis, including Bayesian hierarchical models \cite{Flutre2013} and mixed models \cite{Acharya2016}. Extending such methods to haplotype-based analysis in MPPs poses some challenges, including how to best generalize methods to more complex genetic models and for the CC with a limited number of unique genomes. 
Nonetheless, when multiple levels of molecular traits are measured, joint analyses could conceivably be incorporated into the mediation framework to improve detection power.

\subsection*{Correlated haplotype effects suggest subtle multi-allelic QTL}
Haplotype-based association in MPP allows for the detection of multi-allelic QTL \cite{Crowley2015}, such as potentially observed within the kidney at the local-eQTL of \textit{Pik3c2g}, where mice with B6 contributions in the region have an intermediate level of expression. The correlation coefficient between the founder haplotype effects for QTL pairs provides an interesting summary, generally not possible in the simpler bi-allelic setting commonly used in variant association analysis. The extent of the correlation between the founder effects for QTL pairs of certain genes, such as \textit{Cox7c} (Fig \ref{fig:correlated_local_eqtl}A) suggests that these QTL are at least subtly multi-allelic. Correlated founder effects are consistent with the genetic regulation, even local to the gene TSS, being potentially complex, likely due to founder-specific modifiers.

\subsection*{Correlated haplotype effects with differential expression across tissues}

We detected numerous eQTL pairs with correlated haplotype effects across tissues but with significantly different magnitudes of overall expression---for example, \textit{Cox7c}, \textit{Ubc}, \textit{Slc44a3}, and \textit{Akr1e1}. We propose two potential explanations for this unique co-occurrence. 
First, whole tissues differ in their cellular composition. It may be that these genes are primarily expressed in a common cell type whose proportion varies between the different tissue types, and that the observed differential expression reflects this compositional variation. This hypothesis could be tested by follow-up single cell experiments.
Second, each tissue may have additional unique regulatory elements that further modulate expression levels. Uncovering such elements would require in-depth analysis of tissue-specific regulation.

\subsection*{Mediation analysis}

The statistical methods underlying mediation analyses were largely developed in the context of social sciences \cite{Baron1986,Mackinnon2007,Imai2010}, and more recently extended to a genomics setting in which there is generally less experimental control of the relationship between the mediator and outcome. Our mediation analysis approach is adapted from previous studies in DO mice \cite{Chick2016,Keller2018,Skelly2019} but adds the use of QTL effect sizes to establish consistency with the directionality of the relationships and the formal calculation of an empirical mediation $p$-value through permutation. Related conditional regression approaches were used in the incipient pre-CC lines \cite{Rutledge2014,Kelada2014}.
Mediation results largely reflect the correlations between the variables after adjusting for additional sources of variation, such as covariates and batch effects. We further require the mediator QTL to have a larger effect size than the outcome QTL in order to identify trios that are consistent with the proposed causal models.
It must be emphasized, however, that these steps are not equivalent to experimentally controlling the directionality of the relationship between a gene's expression level and a putative mediator, nor is such control feasible in a large-scale experiment. Variable measurement error on the mediator and gene could flip the perceived directionality of the relationship, resulting in both false positive and negative mediations. Alternatively, the relationships could be more complex than the simple models used here, \eg feedback between the mediator and gene, which these procedures will not detect. Additionally, the causal mediator may not be observed in the data, allowing for other candidates, correlated with the missing causal element, to be incorrectly identified as mediators.

Despite these limitations, the mediation analysis used here provides specific causal candidates for local-eQTL (mediated through chromatin) and distal-eQTL (mediated through nearby genes). For example, we show strong evidence for ZFP985 mediating the genetic regulation of \textit{Akr1e1} (Fig \ref{fig:akr1e1_full_model}). Additional evidence suggests this is done by ZFP985 contributing to reduced \textit{Akr1e1} promoter activity. It is possible that \textit{Zfp985} expression is simply strongly correlated with the true mediator, and others have alternatively proposed \textit{Rex2} expression as a candidate \cite{HamiltonWilliams2010}, which we did not consider due to low expression in all three tissues. Regardless of the identity of the true mediator, our analysis shows strong evidence that it acts causally by reducing chromatin accessibility near \textit{Akr1e1}.

\subsection*{QTL mapping power and their effect size estimates}

Our reduced sample size of 47 CC strains motivated our use of multiple scopes of statistical significance, from testing for QTL locally to genome-wide. As expected, the additional local-QTL detected by less stringent methods have smaller effect sizes (orange and purple dots for C and L, respectively; \nameref{S_qtl_effect_sizes_by_method}). Based on a recent evaluation of QTL mapping power in the CC using simulation \cite{Keele2019}, this study had approximately 80\% power to detect genome-wide QTL with a 55\% effect size or greater. Effect size estimates for genome-wide QTL (green dots; \nameref{S_qtl_effect_sizes_by_method}) are consistent with this expectation albeit potentially inflated due to the Beavis effect \cite{Xu2003}. 
Effect size estimates provide interpretable point summaries for haplotype-based QTL mapping, analogous to minor allele effect estimates in SNP-based studies. We calculated estimates of effect size in two ways, one based on a fixed effect fitting of the QTL term and the other as a random term \cite{Wei2016}. 
%Shrunken effect size estimates can be used to identify putative QTL with unstable fixed effect estimates that result from extreme data points, and thus aid in the identification of low confidence QTL. 
Notably, a small number of the distal-eQTL had low random effect size estimates (\nameref{S_qtl_effect_size_fixefvsranef}) compared with their fixed effects-based estimates, likely the result of outliers with lowly-observed founder inheritance (\eg rare allele) at the putative QTL. Alternatively, a residual variation estimate, \ie RSS, could be calculated from the shrunken haplotype effects to identify likely false positives, but not be as aggressively reduced as the variance component-based estimates, representing a middle ground approach that was found to be effective \cite{Keele2018}.
We primarily report fixed effects estimates due to their consistency with reported expectations \cite{Keele2019} for a study of this size.

These QTL mapping results are largely consistent with the molecular traits with detected QTL possessing primarily Mendelian genetic regulation (large effect sizes: $>$ 60\%). The relatively limited number of CC strains ($<$ 70 strains) constrains our ability to effectively map QTL for highly complex and polygenic traits. Nevertheless, this study supports the value of CC strains for mapping QTL for simpler traits, such as large effect molecular phenotypes, particularly when considering the further gains that use of replicate observations per strain would yield (not used here). Additionally, joint and/or comparative analyses with the DO and the founder strains can provide strong confirmation of subtle findings in the CC.
%\GK{}{I've reduced and rearranged this a bit. Joint/comparative analyses with the DO/founders is a less niche topic to finish on. Initially I had a brief summary, but I think we decided the paper is long enough to skip that.}

\section*{Materials and methods}

\subsection*{Animals}

Adult male mice (8-12 weeks old) from 47 CC strains were acquired from the University of North Carolina Systems Genetics Core (listed in \nameref{S1_Appendix}) and maintained on an NTP 2000 wafer diet (Zeigler Brothers, Inc., Gardners, PA) and water \textit{ad libitum}. The housing room was maintained on a 12-h light-dark cycle. Our experimental design sought to maximize the number of strains relative to within-strain replications based on the power analysis for QTL mapping in mouse populations \cite{Kaeppler1997}; therefore, one mouse was used per strain. Prior to sacrifice, mice were anesthetized with 100 mg/kg nembutal though intraperitoneal injection. Lungs, liver and kidney tissues were collected, flash frozen in liquid nitrogen, and stored at -80\degree C. These studies were approved by the Institutional Animal Care and Use Committees (IACUC) at Texas A\&M University and the University of North Carolina. The experimental design and subsequent analyses performed for this study are diagrammed in Fig \ref{fig:overview}.

% Place figure captions after the first paragraph in which they are cited.

\subsection*{mRNA sequencing and processing}

Total RNA was isolated from flash-frozen tissue samples using a Qiagen miRNeasy Kit (Valencia, CA) according to the manufacturers protocol. RNA purity and integrity were evaluated using a Thermo Scientific Nanodrop 2000 (Waltham, MA) and an Agilent 2100 Bioanalyzer (Santa Clara, CA), respectively. A minimum RNA integrity value of 7.0 was required for RNA samples to be used for library preparation and sequencing. Libraries for samples with a sufficient RNA integrity value were prepared using the Illumina TruSeq Total RNA Sample Prep Kit (Illumina, Inc., San Diego, USA) with ribosomal depletion. Single-end (50 bp) sequencing was performed (Illumina HiSeq 2500).

Sequencing reads were filtered (sequence quality score $\ge$ 20 for $\ge$ 90\% of bases) and adapter contamination was removed (TagDust). Reads were mapped to strain-specific pseudo-genomes (Build37, \url{http://csbio.unc.edu/CCstatus/index.py?run=Pseudo}) and psuedo-transcriptomes (C57BL/6J RefSeq annotations mapped to pseudo-genomes) using RSEM with STAR (v2.5.3a). Uniquely aligned reads were used to quantify expression as transcripts per million (TPM) values.

\subsection*{ATAC-seq processing}

Flash frozen tissue samples were pulverized in liquid nitrogen using the BioPulverizer (Biospec) to break open cells and allow even exposure of intact chromatin to Tn5 transposase \cite{Buenrostro2015}. Pulverized material was thawed in glycerol containing nuclear isolation buffer to stabilize nuclear structure and then filtered through Miracloth (Calbiochem) to remove large tissue debris. Nuclei were washed and directly used for treatment with Tn5 transposase. Paired-end (50 bp) sequencing was performed (Illumina HiSeq 2500).

Reads were similarly filtered as with RNA-seq. Reads were aligned to the appropriate pseudo-genome using GSNAP (parameter set: -k 15, -m 1, -i 5, sampling=1, trim-mismatch-score=0, genome-unk-mismatch=1, query-unk-mismatch=1). Uniquely mapped reads were converted to mm9 (NCBI37) mouse reference genome coordinates using the associated MOD files (UNC) to allow comparison across strains. Reads overlapping regions in the mm9 blacklist (UCSC Genome Browser) were removed. Exact sites of Tn5 transposase insertion were determined as the start position +5 bp for positive strand reads, and the end position -5 bp for negative strand reads \cite{Buenrostro2013}. Peaks were called using F-seq with default parameters. A union set of the top 50,000 peaks (ranked by F-seq score) from each sample was derived. Peaks were divided into overlapping 300 bp windows \cite{Shibata2012}. Per sample read coverage of each window was calculated using coverageBed from BedTools \cite{Quinlan2010}.

\subsection*{Sequence trait filtering for QTL analysis}

Trimmed mean of M-values (TMM) normalization (edgeR; \cite{edgeR}) was applied to TPM values from read counts of genes and chromatin windows, respectively. Genes with TMM-normalized TPM values $\leq$ 1 and chromatin windows with normalized counts $\leq$ 5 for $\geq$ 50\% of samples were excluded (as in \cite{Chick2016}) in order to avoid the detection of QTL that result from highly influential non-zero observations when most of the sample have low to no expression. For each gene and chromatin window, we applied $K$-means clustering with $K=2$ to identify outcomes containing outlier observations that could cause spurious, outlier-driven QTL calls. Any gene or chromatin window where the smaller $K$-means cluster had a cardinality of 1 was removed.

\subsection*{CC strain genotypes and inferred haplotype mosaics}

CC genomes are mosaics of the founder strain haplotypes. The founder haplotype contributions for each CC strain was previously reconstructed by the UNC Systems Genetics Core (\url{http://csbio.unc.edu/CCstatus/index.py?run=FounderProbs}) with a Hidden Markov Model \cite{Fu2012} on genotype calls [MegaMUGA array \cite{Morgan2016muga}] from multiple animals per strain, representing ancestors to the analyzed mice. Notably, QTL mapping power is reduced at loci with segregating variants in these ancestors, and where these specific animals likely differ \cite{Shorter2019}. To reduce the number of statistical tests, adjacent genomic regions were merged through averaging if the founder mosaics for all mice were similar, defined as L2 distance $\leq$ 10\% of the maximum L2 distance ($\sqrt{2}$ for a probability vector). This reduced the number of tested loci from 77,592 to 14,191.

\subsection*{Differential expression and accessibility analyses}

Read counts for each sample were converted to counts per million (CPM), followed by TMM normalization (edgeR). For chromatin accessibility, windows in which $>$ 70 \% of samples had a CPM $\leq$ 1 were removed, requiring that samples from at least two of the three tissues to have non-zero measurements in order to be considered for differential analysis between tissues. Genes and chromatin windows with no or low counts across sample libraries provide little evidence for detection of differential signal, thus removing them reduces the multiple testing burden. Differentially expressed genes and accessible chromatin windows were determined using \textit{limma} \cite{limma}, which fit a linear model of the TMM-normalized CPM value as the response and fixed effect covariates of strain, batch, and tissue (lung, liver, or kidney). 
To account for mean-variance relationships in gene expression and chromatin accessibility data, precision weights were calculated using the \textit{limma} function \textit{voom} and incorporated into the linear modeling procedure. The $p$-values were adjusted using a false discovery rate (FDR) procedure \cite{Benjamini1995}, and differentially expressed genes and accessible chromatin windows were called based on the $q$-value $\le$ 0.01 and log$_{2}$ fold-change $\geq$ 1. Adjacent significantly differential chromatin windows in the same direction were merged with a $p$-value computed using Simes' method \cite{Sarkar1997}, and chromatin regions were re-evaluated for significance using the Simes $p$-values.

\subsection*{Gene set association analysis}

Biological pathways enriched with differentially expressed genes or accessible chromatin were identified with GSAASeqSP \cite{Xiong2014} with Reactome Pathway Database annotations (July 24, 2015 release). A list of assayed genes were input to GSAASeqSP along with a weight for each gene $g$, calculated as:
\begin{equation}
\text{weight}_{g} = \text{sign}(\Delta_{g}) \times (1-q_{g}),
\label{eq:gene_weighting}
\end{equation}
where $\text{sign}(\Delta_{g})$ is the sign of the fold change in gene $g$ expression, and $q_{g}$ is the FDR-adjusted differential expression $p$-value. Pathways with gene sets of cardinality $<$ 15 or $>$ 500 were excluded. 

For pathway analysis of differentially accessible chromatin near genes, each chromatin region was mapped to a gene using GREAT v3.0.0 (\textit{basal plus extension} mode, \textit{5 kb upstream}, \textit{1 kb downstream}, and no distal extension). 
Weights were calculated as with gene expression, but with $\text{sign}(\Delta_{g})$ representing the sign of the fold-change in accessibility of the chromatin region with minimum FDR-adjusted $p$-value that is associated with gene $g$.

\subsection*{Haplotype-based QTL mapping}

QTL analysis was performed for both gene expression and chromatin accessibility using regression on the inferred founder haplotypes \cite{Mott2000}, a variant of Haley-Knott regression \cite{Haley1992,Martinez1992} commonly used for mapping in the CC \cite{Valdar2006c,Aylor2011,Gralinski2015,Kelada2016,Donoghue2017,Keele2019} and other MPPs, such as \textit{Drosophila} \cite{King2012}.

For a given trait---the expression of a gene or the accessibility of a chromatin region---a genome scan was performed in which at each of the 14,191 loci spanning the genome, we fit the linear model,
\begin{equation}
  y_i = \mu + \text{batch}_{b[i]} + \text{QTL}_i + \varepsilon_i\, 
  \label{eq:alternative_model}
\end{equation}
where $y_{i}$ is the trait level for individual $i$, $\mu$ is the intercept, $\text{batch}_b$ is a categorical fixed effect covariate with five levels $b=1,\dots,5$ representing five sequencing batches for both gene expression and chromatin accessibility and where $b[i]$ denotes the batch relevant to $i$, $\varepsilon_i\sim\text{N}(0,\sigma^2)$ is the residual noise, and $\text{QTL}_{i}$ models the genetic effect at the locus, namely that of the eQTL for expression or the cQTL for chromatin accessibility. Specifically, the QTL term models the (additive) effects of alternate haplotype states and is defined as $\text{QTL}_{i}=\bbeta\T\bx_i$, where $\bx_i=(x_{i,\text{AJ}},\dots,x_{i,\text{WSB}})\T$ is a vector of haplotype dosages (\ie, the posterior expected count, from 0 to 2, inferred by the haplotype reconstruction) for the eight founder haplotypes, and $\bbeta=(\beta_\text{AJ},\dots,\beta_\text{WSB})\T$ is a corresponding vector of fixed effects. Note that in fitting this term as a fixed effects vector, the linear dependency among the dosages in $\bx_i$ results in at least one founder haplotype effect being omitted to achieve identifiability; estimation of effects for all eight founders is performed using a modification described below. Prior to model fitting, to avoid sensitivity to non-normality and strong outliers, the response $\{y_i\}^n_{i=1}$ was subject to a rank inverse normal transformation (RINT).
The fit of Eq \ref{eq:alternative_model} was compared with the fit of the same model omitting the QTL term (the null model) by an F-test, leading to a nominal $p$-value, reported as the $\text{logP}=-\log_{10}(\text{$p$-value})$. %A QTL was detected if this logP surpassed a 

\subsection*{QTL detection: local (L), chromosome-wide (C), and genome-wide (G)}

QTL detections were declared according to three distinct protocols of varying stringency and emphasis. The first protocol, termed Analysis L, was concerned only with detection of ``local QTL'', that is, QTL located at or close to the relevant expressed gene or accessible chromatin region. Here we define ``local'' as $\pm$ 10Mb, as been done previously in studies using DO mice \cite{Chick2016}. Our intent is to capture QTL that likely act in \textit{cis} on gene expression and chromatin accessibility, which are expected to have strong effects, while also recognizing the limitations of using haplotype blocks with median size of 16.3 Mb \cite{Srivastava2017} and a small sample size. Fig \nameref{S_dist} suggest that 10Mb generally captures the strongest QTL signals near the gene TSS and chromatin window midpoint.
The second protocol, Analysis C, broadened the search for QTL to anywhere on the chromosome on which the trait is located; the greater number of loci considered meant that the criterion used to call detected QTL for this protocol was more stringent than Analysis L.
The third protocol, Analysis G, further broadened the search to the entire genome with the most stringent detection criterion.
These protocols used two types of multiple test correction: a permutation-based control of the family-wise error rate (FWER) and the Benjamini-Hochberg False Discovery Rate (FDR; \cite{Benjamini1995}). Below we describe in detail the permutation procedure and then the three protocols.

\subsubsection*{Permutation-based error rate control: chromosome- and genome-wide}

The FWER was controlled based on permutations specific to each trait. 1,000 permutations of the sample index were recorded, and then genome scans performed for each trait, using the same permutation orderings. Given a trait, the maximum logP from either the entire genome or the chromosome for each permutation was collected and used to fit a null generalized extreme value distribution (GEV) in order to control the genome- and chromosome-wide error rates, respectively \cite{Dudbridge2004}. 
Error rates were controlled by calculating a $p$-value for each QTL based on the respective cumulative density functions from the GEVs: $\text{permP} = 1 - F_{\text{GEV}}(\text{logP})$, where $F_{\text{GEV}}$ is the cumulative density function of the GEV. We denote genome- and chromosome-wide error rate controlling  $p$-values as $\permpg$ and $\permpc$ respectively.

\WV{The appropriateness of permutation-derived thresholds \cite{Doerge1996} is dependent on the CC having balanced founder haplotype contributions.}{This is ambiguous. Surely there are circumstances where contributions would be balanced but strong genetic similarities between some lines exist.} This assumption was supported by simulations of the funnel breeding design \cite{Valdar2006c}. More recently, power simulations based on the observed CC strain genomes were performed \cite{Keele2019}, which found low levels of population structure for highly polygenic genetic architectures. Nevertheless, we use permutations given that molecular traits often have strong effect QTL that are detectable in the presence of subtle population structure.

\subsubsection*{Local analysis - Analysis L} 
Our detection criteria for local-QTL leverages our strong prior belief in local genetic regulation. For a given trait, this local analysis involved examining QTL associations at all loci within a 10Mb window of the gene TSS or chromatin region midpoint. A QTL was detected if the $\text{permP}_{\text{C}} < 0.05$. Notably, we can also check whether the corresponding $\text{permP}_{\text{G}} < 0.05$, as an additional characterization of statistical significance of detected local-QTL.

\subsubsection*{Chromosome-wide analysis - Analysis C} 
Chromosome-wide analysis involved examining QTL associations at all loci on the chromosome harboring the gene or chromatin region in question. The peak logP is input into a chromosome-wide GEV, producing a $\permpc$, which are further subjected to adjustment by FDR to account for multiple testing across the traits \cite{Chesler2005}. 
Compared with Analysis L, this procedure is more stringent with respect to local-QTL because of the additional FDR adjustment. In addition, it can detect distal-QTL outside the local region of the trait. 
Note, since only the most significant QTL is recorded, this procedure will disregard local-QTL if a stronger distal-QTL on the chromosome is observed.

\subsubsection*{Genome-wide analysis - Analysis G} 
The genome-wide analysis is largely equivalent to Analysis C but examines all genomic loci, while an FDR adjustment is made to the genome-wide $\permpg$. Unlike Analysis C, it incorporates additional scans conditioned on detected QTL to potentially identify multiple QTL per trait (\ie both local- and distal-QTL). Briefly, after a QTL is detected, a subsequent scan (and permutations) is performed in which the previously detected QTL is included in both the null and alternative models, allowing for additional independent QTL to be detected with FDR control. See \nameref{S2_Appendix} for greater detail on this conditional scan procedure and a clear example in Fig \nameref{S_conditional_scan}. Notably, the local/distal status of the QTL does not factor into Analysis G.

Analyses L, C, and G should detect many of the same QTL, specifically strong local-QTL. Collectively, they allow for efficient detection of QTL with varying degrees of statistical support while strongly leveraging local status.

\subsubsection*{QTL effect size}

The effect size of a detected QTL was defined as the $R^2$ attributable to the QTL term; specifically, as $1-\text{RSS}_\text{QTL} / \text{RSS}_0$, where $\text{RSS} = \sum_{i = 1}^{n}(y_{i} - \widehat{y_{i}})^{2}$ is the residual sum of squares, \ie the sum of squares around predicted value $\widehat{y_i}$, and $\text{RSS}_{\text{QTL}}$ and $\text{RSS}_0$ denote the RSS calculated for the QTL and null models respectively. We also calculated a more conservative QTL effect size estimate with a QTL random effects model to compare with the $R^{2}$ estimate.

\subsubsection*{Stable estimates of founder haplotype effects} 

The fixed effects QTL model used for mapping, though powerful for detecting associations, is sub-optimal for providing stable estimates of the haplotype effects vector, $\bbeta$ (calculated as $(\bX^{\text{T}}\bX)^{-1}\bX^{\text{T}}\by$, where $\bX$ is the full design matrix). This is because, among other things, 1) the matrix of haplotype dosages that forms the design matrix of $\{\text{QTL}_i\}^n_{i=1}$ in Eq \ref{eq:alternative_model}, is multi-collinear, which leads to instability, and 2) because the number of observations for some haplotypes will often be few, leading to high estimator variance \cite{Zhang2014}. More stable estimates were therefore obtained using shrinkage. At detected QTL, the model was refit with $\bbeta$ modeled as a random effect, $\bbeta \sim \text{N}(\bzero, \bI\tausq)$ \cite{Wei2016}, to give an 8-element vector of the best linear unbiased estimates (BLUPs; \cite{Robinson1991}), $\blup=(\widetilde{\beta}_\text{AJ},\dots,\widetilde{\beta}_\text{WSB})\T$. These BLUPs, after being centered and scaled, were then used for further comparison of QTL across tissues. 

\subsubsection*{Comparing QTL effects across tissues}

To summarize patterns of the genetic regulation of gene expression and chromatin accessibility across tissues, we calculated correlations between the founder haplotype effects of QTL that map to approximately the same genomic region of the genome for the same traits but in different tissues. For cross-tissue pairs of local-QTL, we required that both be detected within the 20Mb window around the gene TSS or the chromatin window midpoint. For cross-tissue pairs of distal-QTL, the QTL positions had to be within 10Mb of each other. All detected QTL were considered, including QTL from Analyses G and C controlled at FDR $\le 0.2$, allowing for consistent signal across tissues to provide further evidence of putative QTL with only marginal significance in a single tissue.

For a pair of matched QTL $j$ and $k$ from different tissues, we calculated the Pearson correlation coefficient of the BLUP estimates of their haplotype effects, $r_{jk} = \text{cor}(\blup_j, \blup_k)$. Since each $\blup$ is an 8-element vector, the corresponding $r$ are distributed such that $r\sqrt{6}(1 - r^{2})^{-1} \sim t_{6}$ according to the null model of independent variables. Testing alternative models of $r_{jk} > 0$ and $r_{jk} < 0$ produced two $p$-values per pair of QTL. These were then subject to FDR control \cite{Benjamini1995} to give two $q$-values, $q_{ij}^{\{r > 0\}}$ and $q_{ij}^{\{r < 0\}}$. These were then used to classify cross-tissue pairs of QTL as being significantly correlated or anti-correlated, respectively. 

\subsubsection*{Variant association}

Variant association has been used previously in MPP (\eg, \cite{Baud2014, Keele2018}), and uses the same underlying model as the haplotype-based mapping (Eq \ref{eq:alternative_model}), with the QTL term now representing imputed dosages of the minor allele. Variant association can more powerfully detect QTL than haplotype mapping if the simpler variant model is closer to the underlying biological mechanism \cite{Yalcin2005}, though it will struggle to detect multi-allelic QTL.

Variant genotypes from mm10 (NCBI38) were obtained using the ISVdb \cite{Oreper2017} for the CC strains, which were converted to mm9 coordinates with the liftOver tool \cite{Lawrence2009}. Variants were filtered out if their minor allele frequencies $\le 0.1$ or they were not genotyped in one of the CC founder strains to avoid false signals.
Tests of association at individual SNPs (variant association) were performed within the local windows of genes with local-QTL detected in more than one tissue. 
Genes with multiple tissue-selective local-eQTL potentially reflect different functional variants, and could potentially have less consistent patterns of variant association compared with variants that are functionally active in multiple tissues.

\subsection*{Mediation analysis}

Mediation analysis has recently been used with genomic data, including in humans (\eg, \cite{Battle2014,Raulerson2019}) and rodents (\eg, \cite{Keele2018,Oreper2018}), to identify and refine potential intermediates of causal paths underlying phenotypes. We use a similar genome-wide mediation analysis as used with the DO \cite{Chick2016, Keller2018} to detect mediators of eQTL effects on gene expression.

%% Added section that provides little review of classical mediation, and sets up our version

In our study, mediation describes when an eQTL ($X$) appears to act on its target ($Y$) in whole or in part through a third variable ($M$), the mediator, which in this case is a molecular trait. The molecular traits considered here are: 1) chromatin accessibility, in which case $X$ serves also as a cQTL (Fig \ref{fig:graph}A); and, in a separate analysis, 2) the expression of a second, distal, gene, in which case $X$ serves also as a distal-eQTL (Fig \ref{fig:graph}B).
%\WVinline{Traditional mediation analysis \cite{Baron1986} tests whether the data for predefined $\{X,Y,M\}$ are consistent with mediation, doing so in four steps. Steps (1) and (2) establish positive associations $X\rightarrow Y$ and $X\rightarrow M$. Step (3) establishes mediation by examining whether the mediator $M$ explains variation in $Y$ after controling for $X$ (\ie, whether $M\rightarrow Y|X$). Step (4) distinguishes ``full mediation'', where mediation explains the association between $X$ and $Y$ entirely (\ie, $X \indep Y | M$, where ``$\indep$'' denotes independence), from ``partial mediation'', where $Y$ is affected by $X$ both indirectly (through $M$) and directly.}

Traditional mediation analysis \cite{Baron1986} tests whether the data, for predefined $X$, $Y$ and $M$, are consistent with mediation, doing so in four steps. Steps (1) and (2) establish positive associations $X\rightarrow Y$ and $X\rightarrow M$; this corresponds to our requirement that both the transcript $Y$ and the candidate mediator $M$ have a co-localizing QTL $X$. Step (3) establishes mediation by testing for the conditional association $M\rightarrow Y|X$; this corresponds to testing whether the mediator explains variation in gene expression even after controling for the QTL. Step (4) distinguishes ``full mediation'', where mediation explains the association between $X$ and $Y$ entirely, \ie, the QTL acts entirely through the mediator, from ``partial mediation'', where the QTL acts partly through the mediator and partly directly (or through other unmodeled routes).

%\GKinline{In our study, a mediator is a molecular trait, either chromatin accessibility or the expression of another gene, with its own local-QTL that explains a significant portion of the association between an eQTL and the target gene's expression levels Fig \ref{fig:graph}. We require that both the gene outcome and its candidate mediator possess QTL. This is consistent with traditional mediation analysis \cite{Baron1986}, for which the first two steps are: establishing that there are relationships between 1) QTL (X) and gene (Y) and 2) QTL and mediator (M). Subsequent steps then seek to identify and distinguish partial and full mediators, representing M $\rightarrow$ Y $|$ X and X $\indep$ Y $|$ M, respectively, where ``$\indep$'' signifies independence and ``$|$'' signifies conditioning on a variable.}

In practice, evaluating these relationships with sequential hypothesis tests is not straightforward due to multiple testing, and in particular, evaluation of the full mediator relationship does not fit into the traditional testing paradigm. These issues are compounded when analyzing large data from genomics experiments as there are potentially many candidate mediators in close proximity to the QTL. To accommodate the genome-wide nature of the data, we use an empirical approximation to traditional mediation \cite{Chick2016,Keller2018,Skelly2019}: a mediation signal is detected if the strength of the eQTL association ($X\rightarrow Y$) is significantly reduced when the mediator is added as a covariate ($X\rightarrow Y|M$); we do not attempt to distinguish between partial and full mediation since this distinction is too easily obscured by noise.

Candidate mediators of an eQTL were identified using a genome-wide mediation scan described as follows.
%Similar to the QTL mapping genome scan described in Eq \ref{eq:alternative_model}, the mediation scan involves comparing alternative and null models at each potential mediator across the genome.
Consider an eQTL that our previous genome scan has already shown to affect the expression of gene $j$. Denote the expression level for $j$ in individual $i$ as  $y^{\text{G}_{j}}_{i}$ and eQTL effect as $\text{eQTL}_{i}^{\text{G}_{j}}$ (these respectively correspond to $Y$ and $X$ in the mediation description above). Further, consider a proposed mediator $k\in\mathcal{K}$, where $\mathcal{K}$ is the set of all eligible chromatin accessible sites or expressed genes, and let $m_{ik}$ be the value of that mediator for individual $i$. The mediation scan for a given gene/eQTL pair $j$ proceeds by performing, for each proposed mediator $k\in\mathcal{K}$, a model comparison between the alternative model,
\begin{equation}
y^{\text{G}_{j}}_{i} = \mu + \text{eQTL}_{i}^{\text{G}_{j}} + m_{ik} + \text{batch}_{b[i]} + \varepsilon_{i}\,,
\label{eq:mediation_alt}
\end{equation}
and the null model,
\begin{equation}
y^{\text{G}_{j}}_{i} = \mu + m_{ik} \nonumber + \text{batch}_{b[i]} + \varepsilon_{i}\,,
\label{eq:mediation_null}
\end{equation}
where other terms are as described for Eq \ref{eq:alternative_model}.
(Not, shown are additional covariates, such as conditioned loci, which would be included in both the null and alternative models.)
The above model comparison can be seen as re-evaluating the significance of a given eQTL association by conditioning on each proposed mediator in turn (\ie, testing $X\rightarrow Y|M$ for each $M$).
The resulting mediation scan, in contrast to the earlier-described genome scans, thus fixes the QTL location while testing across the genome for candidate mediators.
%hus, in contrast to the QTL genome scan, whereby the QTL effect is fit at positions across the genome, the mediation scan fixes the locus at mthe eQTL, and the mediator term is fit for candidates across the genome.

In general, assuming most proposed mediators are null, the mediated logP should fluctate around the original eQTL logP, since the model comparison will resemble the original test of that locus in the genome scan.
For mediators that possess some or all of the information present in the eQTL, however, the mediated logP will drop relative to the original logP, reflecting $X\rightarrow Y|M$ being less significant than  $X\rightarrow Y$. 
Empirical significance of genome-wide mediation has previously been determined by comparing the nominal mediator scores to the distribution of mediation scores genome-wide, the latter effectively acting as a null distribution \cite{Chick2016,Keller2018,Skelly2019}. We instead determine a null distribution explicitly by permutation, characterizing the distribution of the minimum logP (as opposed to maximum logP for QTL scans) to estimate significance and set false positive control (FWER).
As with the QTL mapping procedures, we performed mediation scans on 1,000 permutations%
, permuting the mediators rather than the outcome, to characterize GEVs from the minimum logP, that is, $\permpmed = F_{\text{GEV}}(\text{logP})$, at both chromosome- and genome-wide levels. 

Detection of mediation is dependent on a number of assumptions about the underlying variables, their relationships, and those relationships' directionality \cite{Mackinnon2007}. Many of these cannot be controlled in a system as complex as chromatin accessibility and transcriptional regulation in whole living organisms. Nevertheless, signals in the data that are consistent with the mediation model represent candidate causal factors that regulate gene expression.
Though it is impossible to ensure that the \WVinline{direction of causation is indeed $M\rightarrow Y$, as assumed by the mediation analysis,} we require additional checks to formally declare mediation of an eQTL. The presence of the relationship $X \rightarrow Y$ is already established in that mediation scans are only performed for detected eQTL. In addition for any candidate mediator with a significant $\permpmed$, we require detection of a mediator QTL: $X \rightarrow M$. These requirements, as mentioned earlier, are consistent with traditional mediation analysis. Finally, in an attempt to identify mediator-to-gene relationships consistent with the proposed models, we require that $X \rightarrow M $ is more significant than $ X \rightarrow Y$. Though this step cannot confirm the directionality of the relationship between $M$ and $Y$, as variable noise level between $M$ and $Y$ could reduce the estimated effect size of their respective QTL, it will identify candidate mediators that are consistent with the proposed models. Further details on the mediation analysis are described in \nameref{S3_Appendix}, including the permutation approach and the formal criterion by which mediation is declared for both chromatin and gene mediators (Fig \ref{fig:graph}).

\subsection*{Software}

All statistical analyses were conducted with the R statistical programming language \cite{RSoftware2019}. The R package miQTL was used for all the mapping and mediation analyses, and is available on GitHub at \url{https://github.com/gkeele/miqtl}.

\section*{Supporting information}

% Include only the SI item label in the paragraph heading. Use the \nameref{label} command to cite SI items in the text.
\paragraph*{S1 Fig.}
\label{S_pca}
{\bf Principle components analysis identifies tissue type as key source of variation for gene expression and chromatin accessibility.} 
Molecular traits for liver (purple), lung (green), and kidney (orange) tissue samples were derived from RNA-seq and ATAC-seq data. Principal components (PC) 1 and 2 capture a majority of the variation and show a greater amount of between tissue variability than within tissue variability. (PDF)

\paragraph*{S2 Fig.}
\label{S_de_concordance}
{\bf Concordance between differentially expressed genes and differentially accessible regions in between-tissue comparisons.} 
Genes were categorized by the direction of the difference in expression and chromatin accessibility in their promoter regions. (PDF)

\paragraph*{S3 Fig.}
\label{S_upset_outcomes}
{\bf Overlap across tissues of (A) genes and (B) chromatin windows used for QTL analysis.} 
Sequence traits were filtered to remove outcomes more likely to cause spurious QTL signals. Genes with TPM $\le 1$ and chromatin windows with TMP $\le 5$ for $\ge$ 50\% of samples were removed from analysis. After this filtering process, lung had the greatest number of traits analyzed, for both genes and chromatin windows, followed by kidney and then liver. (PDF)

\paragraph*{S4 Fig.}
\label{S_upset_qtl}
{\bf Overlap across tissues of (A) genes and (B) chromatin windows with local-QTL detected.} 
The majority of sequence traits with a local-QTL detected were identified in only a single tissue. Kidney had the highest number of local-eQTL, whereas lung had the highest number of local-cQTL. Liver had a relative lack of local-cQTL, which may relate to its having the fewest chromatin windows analyzed (\nameref{S_upset_outcomes}B). Results included local-QTL detected with Analysis G (FDR $\le 0.1$), Analysis C (FDR $\le 0.1$), and Analysis L (genome-wide and chromosome-wide). (PDF)

\paragraph*{S5 Fig.}
\label{S_grid_plot_lenient}
{\bf QTL mapping results using only Analysis G or Analysis C.}
QTL map plots of (A) eQTL and (B) cQTL with FDR controlled at 0.1 and 0.2 for liver, lung, and kidney. Detected QTL from Analysis G (multi-stage FDR) and Analysis C (chromosome-wide FDR) are included. Analysis C, which uses FDR control for chromosome-wide significant QTL, produces a large number of intra-chromosomal distal QTL. The y-axis represents the genomic position of the gene or chromatin site, and the x-axis represents the genomic position of the QTL. Local-QTL appear as dots along the diagonal. (PDF)

\paragraph*{S6 Fig.}
\label{S_dist}
{\bf Highly significant QTL map nearby the gene TSS and chromatin window midpoint.}
The permutation-based $p$-value (permP) from (A) Analysis G and (B) Analysis C for eQTL and cQTL by their distance (Mb) from the gene TSS and the midpoint of the chromatin site. Inter-chromosomal distal-QTL are not included. The red dashed lines represent $\pm$ 10Mb of the gene TSS or the midpoint of the chromatin site for classifying QTL as local or distal. Significant signals (yellow or blue), based on FDR $\le 0.1$, are largely local. Analysis C detects many more intra-chromosomal distal-QTL. (PDF)

\paragraph*{S7 Fig.}
\label{S_effect_size_status}
{\bf QTL effect size by local/distal status.}
Each dot represents a QTL detected through either (A) Analysis G or (B) Analysis C with FDR $\le 0.1$. The three horizontal bars represent the 25\textsuperscript{th}, 50\textsuperscript{th}, and 75\textsuperscript{th} quantiles of QTL effect sizes for all local-QTL per tissue. More local-eQTL are detected and have higher effects than distal-QTL. Analysis C detects a large number of intra-chromosomal distal-QTL that Analysis G does not, many of which have low effect sizes. Effect size estimates are based on a fixed effects model. (PDF)

\paragraph*{S8 Fig.}
\label{S_qtl_effects_abs}
{\bf CAST and PWK haploytpes have more extreme haplotype effects for (A) eQTL and (B) cQTL compared with the other strains.}
Haplotype effects were estimated as BLUPs, which are constrained and centered around 0. Each QTL is represented by an 8-element effect vector. Founders with more extreme effects are identified by comparing the absolute values of effects. Founder haploytpe effect trends for eQTL are similar to cQTL. The trends are unstable in distal-cQTL because so few are identified. (PDF)

\paragraph*{S9 Fig.}
\label{S_qtl_effect_size_comparison}
{\bf Effect sizes between cross-tissue QTL pairs are lowly but significantly correlated.}
Comparisons of QTL effects sizes between (liver/lung) are in the left column, (liver/kidney) middle column, and (lung/kidney) right column. eQTL are yellow and cQTL are blue. Local-eQTL are plotted in the top row, distal-eQTL in the second row, local-cQTL in the third row, and distal-cQTL in the bottom row, with only four pairs detected in (lung/kidney). (PDF)

\paragraph*{S10 Fig.}
\label{S_qtl_pair_histograms}
{\bf Consistent genetic regulation of gene expression and chromatin accessibility observed across tissues.}
There was an excess of significant positively correlated founder haplotype effects in QTL pairs across tissues for gene expression and chromatin accessibility. Pairs of QTL observed in multiple tissues were defined for local-eQTL (left column), distal-eQTL (middle column), and local-cQTL (right column). Only four pairs of distal-cQTL were observed, all shared between lung and kidney. A right-tailed test the correlation between founders effects ($H_{A}: r > 0$) was performed for each QTL pair, producing $p$-values that were then FDR adjusted. Null simulations of uncorrelated 8-element vector pairs for each class of QTL and pairwise tissue comparison emphasize the observed enrichment in correlated founder effects between QTL pairs. (PDF)

\paragraph*{S11 Fig.}
\label{S_qtl_cor_by_distance_comparison}
{\bf Cross-tissue QTL pairs with highly correlated founder haplotype effects map proximally to each other.}
Founder effects were estimated as constrain BLUPs. Pairwise correlations of the 8-element effect vectors were calculated for QTL pairs, and plotted again the distance between the QTL coordinates in Mb for (liver/lung) in the left column, (liver/kidney) in the middle column, and (lung/kidney) in the right column. Single eQTL and cQTL pairs are represented as a yellow and blue dots, respectively. Local-eQTL are shown in the top row, distal-eQTL in the second row, local-cQTL in the third row, and distal c-QTL in the bottom row, for which only four pairs were detected in (lung/kidney). (PDF)

\paragraph*{S12 Fig.}
\label{S_ubc_correlated_eqtl}
{\bf The gene \textit{Ubc} has consistent strong local-eQTL observed in the three tissues.}
The local-eQTL consistently drove higher expression when the B6, NOD, NZO, and WSB haploytpes were present. Expression levels in liver and lung were found to be significantly different ($q = 0.022$). The estimated haplotype effects, scaled BLUPs, were highly consistent with the expression data, represented as interquartile bars by most likely diplotype. The haplotype and variant associations in the eQTL regions were similar across tissues, suggesting they may represent the same causal origin. The red tick represents the \textit{Ubc} TSS, the black tick represents the peak variant association, and the colored ticks represent the peak haplotype association, each each tissue. (PDF)

\paragraph*{S13 Fig.}
\label{S_rnf13_distal_eqtl}
{\bf The gene \textit{Rnf13} has tissue-selective patterns of genetic regulation.}
A strong local-eQTL detected in liver. After conditioning on the local-eQTL, a statistically significant (Analysis G) distal-eQTL was detected on chromosome 12, largely driven by the B6 haplotype, distinct from the local-eQTL. The unique haplotype effect patterns for each eQTL can be seen in both the expression data, represented by interquartile bars by most likely diplotype, and the estimated effects (scaled BLUPs). On the association scans, the red tick marks the \textit{Rnf13} TSS and the black tick marks the location of distal-eQTL. (PDF)

\paragraph*{S14 Fig.}
\label{S_colocalization}
{\bf Co-localizing eQTL and cQTL are not sufficient for statistical mediation.}
The approach used to detect mediation through chromatin accessibility requires that the eQTL and cQTL co-localize (both within 10Mb of the gene TSS), as well as possess similar founder haplotype effect patterns. Co-localizing cQTL are observed for local-eQTL for both (A) \textit{Hdhd3} in liver and (B) \textit{Acbd4} in kidney. QTL and mediation scans are shown, with chromosomes 4 and 11 blown up for \textit{Hdhd3} and \textit{Acbd4}, respectively. The red ticks denote the TSS for both genes. The founders effects were estimated as centered and scaled BLUPs. The founder effects for the eQTL and cQTL are highly correlated ($r = 0.96$) for \textit{Hdhd3}, but not for \textit{Acbd4} ($r = 0.52$). Strong mediation of the \textit{Hdhd3} eQTL through chromatin is detected, but not for \textit{Acbd4}. The effect size of the proximal cQTL to \textit{Acbd4} is smaller than its eQTL, also inconsistent with the relationship depicted in Fig 6A. (PDF)

\paragraph*{S15 Fig.}
\label{S_ccnyl1_exmediation}
{\bf Mediation of \textit{Ccnyl1} distal-eQTL through \textit{Zfp979} expression.}
Expression of \textit{Ccnyl1} and \textit{Zfp979} are correlated ($r = 0.72$) in lung, which is also observed in the expression data categorized by diplotype and the founder effects estimated as scaled BLUPs. The distal-eQTL on chromosome 4 for \textit{Ccnyl1} corresponds closely to local-eQTL of \textit{Zfp979}. \textit{Ccnyl1} is located on chromosome 1, indicated by the red tick. \textit{Zfp979} and \textit{Zfp985}, both zinc finger proteins likely with DNA binding properties, are identified as strong candidate mediators of the distal-eQTL at genome-wide significance. The correlations, magnitude of effects, and mediation are consistent with the simple relationship depicted in the graph. The distal-eQTL and candidate mediators are located in a region of interest that regulates \textit{Akr1e1} expression. (PDF)

\paragraph*{S16 Fig.}
\label{S_akr1e1_exmediation}
{\bf Mediation of \textit{Akr1e1} distal-eQTL through \textit{Zfp985} expression.}
Expression of \textit{Akr1e1} and \textit{Zfp985} are anti-correlated ($r = -0.69$) in lung. This relationship is also observed in the expression data with bars representing the interquartile range, categorized by most likely diplotype, and the founder haplotype effects, estimated as scaled BLUPs. The QTL and mediation scans reveal that \textit{Akr1e1}, TSS marked with a red tick on chromosome 13, possesses a distal-eQTL on chromosome 4 that is proximal to the strong local-eQTL of \textit{Zfp985}. The mediation scan identifies \textit{Zfp985} as a strong candidate mediator consistent with the mediation model. A more complete picture of the genetic regulation of \textit{Akr1e1} expression is pieced together by looking across all three tissues and includes a potential chromatin mediator (Fig 9). (PDF)

\paragraph*{S17 Fig.}
\label{S_do_akr1e1}
{\bf Confirmation of \textit{Akr1e1} distal-eQTL and mediation by \textit{Zfp985} in kidney tissue of Diversity Outbred mice.}
A genome-wide significant distal-eQTL was detected for \textit{Akr1e1} in liver, lung (shown here), and kidney tissues from 47 CC strains. In a larger sample of kidney tissue from outbred DO mice, the same distal-eQTL and mediation relationship were observed. As expected, the larger sample results in greater statistical significance, and confirms that the NOD effect is more strongly negative than NZO, PWK, and WSB, which the effects plots the \textit{Zfp985} local-eQTL suggested. Notably, \textit{Zfp985} was not tested in the CC kidney because of low levels, though the distal-eQTL for \textit{Akr1e1} is consistent with its activity, which is here confirmed in the DO. (PDF)

\paragraph*{S18 Fig.}
\label{S_akr1e1_relationships}
{\bf Observed relationships across the three tissues related to the genetic regulation of \textit{Akr1e1} expression.}
The model for the distal genetic regulation of \textit{Akr1e1} expression, described in Fig 9, was reconstructed from these observed relationships. Solid arrows were observed, whereas dashed arrows are assumed. Effect sizes represent the proportion of variance explained by the QTL and mediation $p$-values (permP) were defined using a permutation procedure. The assumed relationships are supported by the presence of the distal-eQTL in all three tissues. The \textit{Zfp985} mediator relationship in kidney, though not observed in the CC, was observed in the related DO population. (PDF)

\paragraph*{S19 Fig.}
\label{S_qtl_effect_sizes_by_method}
{\bf Local-QTL effect sizes by mapping analysis.}
Based on ranking mapping analyses with respect to the extent of scope, local (L; magenta) to chromosome (C; plum) to genome-wide (G; cyan), the greater the scope corresponded to reduced power to detect QTL, shown in liver, lung, and kidney tissues for gene expression (yellow line) and chromatin accessibility (blue line). Each dot represents a detected local-QTL, colored according to the highest scope mapping procedure that detected it. The three horizontal bars represent the 25\textsuperscript{th}, 50\textsuperscript{th}, and 75\textsuperscript{th} quantiles of QTL effect sizes for all local-QTL per tissue. Analysis G generally detects QTL with effect size $>$ 60\%, whereas Analyses C and L detect QTL effect sizes $>$ 45\%. Effect size estimates correspond to a fixed effects model of the QTL. (PDF)

\paragraph*{S20 Fig.}
\label{S_qtl_effect_size_fixefvsranef}
{\bf Comparison of QTL effect sizes estimates from fixed effects and random effects models.}
The effect size corresponding to the random effect fit is harshly penalized compared with the fixed effect estimate, likely due to a small sample size of 47 individuals. Notably, there are a number of distal-eQTL that are more harshly reduced by the random effects model compared with the other QTL, likely representing signals resulting from extreme observations or imbalances in founder contributions at the locus. QTL detected by Analysis G (FDR $\le 0.1$), 2 (FDR $\le 0.1$), and 3 are shown. (PDF)

\paragraph*{S21 Fig.}
\label{S_conditional_scan}
{\bf Detection of local-eQTL after conditioning on distal-eQTL for \textit{Gpn3}.} 
The multi-stage conditional regression approach of Analysis G allows for the detection of multiple genome-wide significant QTL, which can be appropriately incorporated into a FDR procedure across many outcomes. In this example in lung tissue, the gene \textit{Gpn3} initially has a strong distal-eQTL on chromosome 8 [top left]. Though a peak is detected near the TSS of \textit{Gpn3}, it does not meet genome-wide significance. However, after conditioning on the distal-eQTL, the local-eQTL is detected [bottom left]. Horizontal dashed lines represent empirical 95\% significance thresholds based on 1,000 permutations. (PDF)


%\paragraph*{S1 File.}
%\label{S1_File}
%{\bf Lorem ipsum.}  Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

\paragraph*{S1 Appendix.}
\label{S1_Appendix}
{\bf CC strains used in study.} 
(PDF)

\paragraph*{S2 Appendix.}
\label{S2_Appendix}
{\bf Detailed description of conditional genome-wide scans (Analysis G).}
(PDF)

\paragraph*{S3 Appendix.}
\label{S3_Appendix}
{\bf Detailed description of mediation analysis.}
(PDF)

\paragraph*{S1 Table.}
\label{S_de}
{\bf Number of differentially expressed genes and accessible chromatin regions detected between liver, lung, and kidney tissues.} 
(PDF)

\paragraph*{S2 Table.}
\label{S_eqtl}
{\bf Number of genes with eQTL detected in liver, lung, and kidney tissues at FDR $\le 0.1$.} 
(PDF)

\paragraph*{S3 Table.}
\label{S_eqtl_lenient}
{\bf Number of genes with eQTL detected in liver, lung, and kidney tissues at FDR $\le 0.2$.} 
(PDF)

\paragraph*{S4 Table.}
\label{S_cqtl}
{\bf Number of chromatin accessibility sites with cQTL detected in liver, lung, and kidney tissues at FDR $\le 0.1$.} 
(PDF)

\paragraph*{S5 Table.}
\label{S_cqtl_lenient}
{\bf Number of chromatin accessibility sites with cQTL detected in liver, lung, and kidney tissues at FDR $\le 0.2$.} 
(PDF)

\paragraph*{S6 Table.}
\label{S_mediation}
{\bf Number of genes with chromatin mediation of local-eQTL in liver, lung, and kidney tissues.} 
(PDF)

\paragraph*{S7 Table.}
\label{S_exmediation}
{\bf Genes with distal-eQTL with gene mediators detected in lung and kidney tissues.} 
(PDF)


\section*{Acknowledgments}
This work was funded by grants from the National Institute of Environmental Health Sciences (NIEHS) (R01-ES023195 to IR and GEC and P30-ES025128), with GRK and WV funded by grants from the National Institute of General Medical Sciences (NIGMS) (R01-GM104125 and R35-GM127000 to WV). From UNC, we thank Lucas T.\ Laudermilk for his insights on the functional mechanisms of zinc finger protein-induced gene silencing, and Samir N.\ P.\ Kelada and Lauren J.\ Donoghue for reading the manuscript and providing thoughtful suggestions. We thank Fred A.\ Wright of North Carolina State University for helpful discussions on eQTL and mediation analyses. We thank Gary A.\ Churchill of The Jackson Laboratory for discussions on mediation analysis that influenced this work.
\section*{Author Contributions}

\noindent \textbf{Conceptualization:} Ivan Rusyn, Terrence S. Furey, Gregory E. Crawford, Gregory R. Keele, William Valdar

\noindent \textbf{Data Curation:} Jennifer W. Israel, Jeremy M. Simon, Paul Cotney

\noindent \textbf{Formal Analysis:} Gregory R. Keele, Bryan C. Quach

\noindent \textbf{Funding Acquisition:} Terrence S. Furey, Ivan Rusyn, Gregory E. Crawford

\noindent \textbf{Investigation:} Grace A. Chappell, Lauren Lewis, Alexias Safi,

\noindent \textbf{Methodology:} Gregory R. Keele, William Valdar

\noindent \textbf{Project Admin:} Terrence S. Furey, Ivan Rusyn, Gregory E. Crawford, William Valdar

\noindent \textbf{Software:} Gregory R. Keele, Bryan C. Quach

\noindent \textbf{Supervision:} Terrence S. Furey, Ivan Rusyn, William Valdar, Gregory E. Crawford

\noindent \textbf{Visualization:} Gregory R. Keele, Bryan C. Quach

\noindent \textbf{Writing  Original:} Gregory R. Keele, Bryan C. Quach

\noindent \textbf{Writing  Review:} Terrence S. Furey, William Valdar, Ivan Rusyn
 
\nolinenumbers

% Either type in your references using
% \begin{thebibliography}{}
% \bibitem{}
% Text
% \end{thebibliography}
%
% or
%
% Compile your BiBTeX database using our plos2015.bst
% style file and paste the contents of your .bbl file
% here. See http://journals.plos.org/plosone/s/latex for 
% step-by-step instructions.
% 
%\bibliography{eQTL_mediation}

\begin{thebibliography}{10}

\bibitem{Schadt2009}
Schadt EE.
\newblock {Molecular networks as sensors and drivers of common human diseases}.
\newblock Nature. 2009;461(7261):218--223.
\newblock doi:{10.1038/nature08454}.

\bibitem{Schaid2018}
Schaid DJ, Chen W, Larson NB.
\newblock {From genome-wide associations to candidate causal variants by
  statistical fine-mapping}.
\newblock Nature Reviews Genetics. 2018;19(8):491--504.
\newblock doi:{10.1038/s41576-018-0016-z}.

\bibitem{Baron1986}
Baron RM, Kenny DA.
\newblock {The moderator-mediator variable distinction in social psychological
  research: Conceptual, strategic, and statistical considerations.}
\newblock Journal of Personality and Social Psychology. 1986;51(6):1173--1182.
\newblock doi:{10.1037/0022-3514.51.6.1173}.

\bibitem{Mackinnon2007}
MacKinnon DP, Fairchild AJ, Fritz MS.
\newblock {Mediation Analysis}.
\newblock Annual Review of Psychology. 2007;58(1):593--614.
\newblock doi:{10.1146/annurev.psych.58.110405.085542}.

\bibitem{Imai2010}
Imai K, Keele L, Yamamoto T.
\newblock {Mediation Analysis}.
\newblock Statistical Science. 2010;25(1):51--71.
\newblock doi:{10.1214/10-STS321}.

\bibitem{Richmond2016}
Richmond RC, Hemani G, Tilling K, {Davey Smith} G, Relton CL.
\newblock {Challenges and novel approaches for investigating molecular
  mediation}.
\newblock Human Molecular Genetics. 2016;25(R2):R149--R156.
\newblock doi:{10.1093/hmg/ddw197}.

\bibitem{Degner2012}
Degner JF, Pai AA, Pique-Regi R, Veyrieras JB, Gaffney DJ, Pickrell JK, et~al.
\newblock {DNase I sensitivity QTLs are a major determinant of human expression
  variation.}
\newblock Nature. 2012;482(7385):390--4.
\newblock doi:{10.1038/nature10808}.

\bibitem{Pai2015}
Pai AA, Pritchard JK, Gilad Y.
\newblock {The genetic and mechanistic basis for variation in gene regulation.}
\newblock PLoS Genetics. 2015;11(1):e1004857.
\newblock doi:{10.1371/journal.pgen.1004857}.

\bibitem{Battle2015}
Battle A, Khan Z, Wang SH, Mitrano A, Ford MJ, Pritchard JK, et~al.
\newblock {Genomic variation. Impact of regulatory variation from RNA to
  protein.}
\newblock Science (New York, NY). 2015;347(6222):664--7.
\newblock doi:{10.1126/science.1260793}.

\bibitem{Yang2017}
Yang F, Wang J, {GTEx Consortium}, Pierce BL, Chen LS.
\newblock {Identifying cis-mediators for trans-eQTLs across many human tissues
  using genomic mediation analysis.}
\newblock Genome Research. 2017;27(11):1859--1871.
\newblock doi:{10.1101/gr.216754.116}.

\bibitem{Battle2014}
Battle A, Mostafavi S, Zhu X, Potash JB, Weissman MM, McCormick C, et~al.
\newblock {Characterizing the genetic basis of transcriptome diversity through
  RNA-sequencing of 922 individuals}.
\newblock Genome Research. 2014;24(1):14--24.
\newblock doi:{10.1101/gr.155192.113}.

\bibitem{Alasoo2017}
Alasoo K, Rodrigues J, Mukhopadhyay S, Knights AJ, Mann AL, Kundu K, et~al.
\newblock {Shared genetic effects on chromatin and gene expression indicate a
  role for enhancer priming in immune response.}
\newblock Nature Genetics. 2018; p. 102392.
\newblock doi:{10.1038/s41588-018-0046-7}.

\bibitem{Roytman2018}
Roytman M, Kichaev G, Gusev A, Pasaniuc B.
\newblock {Methods for fine-mapping with chromatin and expression data.}
\newblock PLoS Genetics. 2018;14(2):e1007240.
\newblock doi:{10.1371/journal.pgen.1007240}.

\bibitem{Wu2018}
Wu Y, Zeng J, Zhang F, Zhu Z, Qi T, Zheng Z, et~al.
\newblock {Integrative analysis of omics summary data reveals putative
  mechanisms underlying complex traits}.
\newblock Nature Communications. 2018;9(1):918.
\newblock doi:{10.1038/s41467-018-03371-0}.

\bibitem{Raulerson2019}
Raulerson CK, Ko A, Kidd JC, Currin KW, Brotman SM, Cannon ME, et~al.
\newblock {Adipose Tissue Gene Expression Associations Reveal Hundreds of Candidate Genes for Cardiometabolic Traits}.
\newblock The American Journal of Human Genetics. 2019.
\newblock doi:{10.1016/j.ajhg.2019.09.001}.

\bibitem{Visscher2017}
Visscher PM, Wray NR, Zhang Q, Sklar P, McCarthy MI, Brown MA, et~al.
\newblock {10 Years of GWAS Discovery: Biology, Function, and Translation.}
\newblock American Journal of Human Genetics. 2017;101(1):5--22.
\newblock doi:{10.1016/j.ajhg.2017.06.005}.

\bibitem{Churchill2004}
Churchill GA, Airey DC, Allayee H, Angel JM, Attie AD, Beatty J, et~al.
\newblock {The Collaborative Cross, a community resource for the genetic
  analysis of complex traits}.
\newblock Nature Genetics. 2004;36(11):1133--1137.
\newblock doi:{10.1038/ng1104-1133}.

\bibitem{Hall2012}
{Collaborative Cross Consortium}.
\newblock {The genome architecture of the Collaborative Cross mouse genetic
  reference population.}
\newblock Genetics. 2012;190(2):389--401.
\newblock doi:{10.1534/genetics.111.132639}.

\bibitem{Srivastava2017}
Srivastava A, Morgan AP, Najarian ML, Sarsani VK, Sigmon JS, Shorter JR, et~al.
\newblock {Genomes of the mouse collaborative cross}.
\newblock Genetics. 2017;206(2):537--556.
\newblock doi:{10.1534/genetics.116.198838}.

\bibitem{Churchill2012}
Churchill G, Gatti D, Munger S, Svenson K.
\newblock {The Diversity outbred mouse population}.
\newblock Mammalian Genome. 2012;23:713--8.
\newblock doi:{10.1007/s00335-012-9414-2.The}.

\bibitem{DeKoning2017}
de Koning DJ, McIntyre LM.
\newblock {Back to the Future: Multiparent Populations Provide the Key to Unlocking the Genetic Basis of Complex Traits}.
\newblock G3 (Bethesda, Md). 2017;7(6):1617--1618.
\newblock doi:{10.1534/genetics.117.203265}.

\bibitem{Chick2016}
Chick JM, Munger SC, Simecek P, Huttlin EL, Choi K, Gatti DM, et~al.
\newblock {Defining the consequences of genetic variation on a proteome-wide
  scale.}
\newblock Nature. 2016;534(7608):500--5.
\newblock doi:{10.1038/nature18270}.

\bibitem{Skelly2019}
Skelly DA, Czechanski A, Byers C, Aydin S, Spruce C, Olivier C, et~al.
\newblock Genetic variation influences pluripotent ground state stability in
  mouse embryonic stem cells through a hierarchy of molecular phenotypes.
\newblock bioRxiv. 2019;doi:{10.1101/552059}.

\bibitem{Zhang2014}
Zhang Z, Wang W, Valdar W.
\newblock {Bayesian modeling of haplotype effects in multiparent populations.}
\newblock Genetics. 2014;198(1):139--56.
\newblock doi:{10.1534/genetics.114.166249}.

\bibitem{Kaeppler1997}
Kaeppler SM.
\newblock {Quantitative trait locus mapping using sets of near-isogenic lines:
  Relative power comparisons and technical considerations}.
\newblock Theoretical and Applied Genetics. 1997;95(3):384--392.
\newblock doi:{10.1007/s001220050574}.

\bibitem{Buenrostro2015}
Buenrostro JD, Wu B, Chang HY, Greenleaf WJ.
\newblock {ATAC-seq: A Method for Assaying Chromatin Accessibility
  Genome-Wide}.
\newblock In: Current Protocols in Molecular Biology. vol. 2015. Hoboken, NJ,
  USA: John Wiley {\&} Sons, Inc.; 2015. p. 21.29.1--21.29.9.
\newblock Available from:
  \url{http://doi.wiley.com/10.1002/0471142727.mb2129s109}.

\bibitem{Buenrostro2013}
Buenrostro JD, Giresi PG, Zaba LC, Chang HY, Greenleaf WJ.
\newblock {Transposition of native chromatin for fast and sensitive epigenomic
  profiling of open chromatin, DNA-binding proteins and nucleosome position.}
\newblock Nature Methods. 2013;10(12):1213--8.
\newblock doi:{10.1038/nmeth.2688}.

\bibitem{Shibata2012}
Shibata Y, Sheffield NC, Fedrigo O, Babbitt CC, Wortham M, Tewari AK, et~al.
\newblock {Extensive evolutionary changes in regulatory element activity during
  human origins are associated with altered gene expression and positive
  selection.}
\newblock PLoS Genetics. 2012;8(6):e1002789.
\newblock doi:{10.1371/journal.pgen.1002789}.

\bibitem{Quinlan2010}
Quinlan AR, Hall IM.
\newblock {BEDTools : a flexible suite of utilities for comparing genomic
  features}. 2010;26(6):841--842.
\newblock doi:{10.1093/bioinformatics/btq033}.

\bibitem{edgeR}
Robinson MD, McCarthy DJ, Smyth GK.
\newblock edgeR: a Bioconductor package for differential expression analysis of
  digital gene expression data.
\newblock Bioinformatics. 2010;26(1):139--140.

\bibitem{Fu2012}
Fu CP, Welsh CE, de~Villena FPM, McMillan L.
\newblock {Inferring ancestry in admixed populations using microarray probe
  intensities}.
\newblock In: Proceedings of the ACM Conference on Bioinformatics,
  Computational Biology and Biomedicine - BCB '12. New York, New York, USA: ACM
  Press; 2012. p. 105--112.
\newblock Available from:
  \url{http://dl.acm.org/citation.cfm?id=2382936.2382950
  http://dl.acm.org/citation.cfm?doid=2382936.2382950}.

\bibitem{Morgan2016muga}
Morgan AP, Fu CP, Kao CY, Welsh CE, Didion JP, Yadgary L, et~al.
\newblock {The Mouse Universal Genotyping Array: From Substrains to
  Subspecies}.
\newblock G3 (Bethesda, Md). 2016;6(2):263--279.
\newblock doi:{10.1534/g3.115.022087}.

\bibitem{Shorter2019}
Shorter JR, Najarian ML, Bell TA, Blanchard M, Ferris MT, Hock P, et~al.
\newblock Whole Genome Sequencing and Progress Towards Full Inbreeding of the
  Mouse Collaborative Cross Population.
\newblock G3: Genes, Genomes, Genetics. 2019;doi:{10.1534/g3.119.400039}.

\bibitem{limma}
Ritchie ME, Phipson B, Wu D, Hu Y, Law CW, Shi W, et~al.
\newblock {limma} powers differential expression analyses for {RNA}-sequencing
  and microarray studies.
\newblock Nucleic Acids Research. 2015;43(7):e47.

\bibitem{Benjamini1995}
Benjamini Y, Hochberg Y
\newblock {Controlling the False Discovery Rate : A Practical and Powerful
  Approach to Multiple Testing}.
\newblock Journal of the Royal Statistical Society, Series B.
  1995;57(1):289--300.

\bibitem{Sarkar1997}
Sarkar SK, Chang CK.
\newblock {The Simes Method for Multiple Hypothesis Testing With Positively
  Dependent Test Statistics}.
\newblock Journal of the American Statistical Association. 1997;92(440):1601.
\newblock doi:{10.2307/2965431}.

\bibitem{Xiong2014}
Xiong Q, Mukherjee S, Furey TS.
\newblock {GSAASeqSP: a toolset for gene set association analysis of RNA-Seq
  data.}
\newblock Scientific Reports. 2014;4:6347.
\newblock doi:{10.1038/srep06347}.

\bibitem{Mott2000}
Mott R, Talbot CJ, Turri MG, Collins AC, Flint J.
\newblock {A method for fine mapping quantitative trait loci in outbred animal
  stocks.}
\newblock Proceedings of the National Academy of Sciences.
  2000;97(23):12649--54.
\newblock doi:{10.1073/pnas.230304397}.

\bibitem{Haley1992}
Haley CS, Knott SA.
\newblock {A simple regression method for mapping quantitative trait loci in
  line crosses using flanking markers}.
\newblock Heredity. 1992;69:315--24.

\bibitem{Martinez1992}
Mart{\'{i}}nez O, Curnow RN.
\newblock {Estimating the locations and the sizes of the effects of
  quantitative trait loci using flanking markers}.
\newblock Theoretical and Applied Genetics. 1992;85(4):480--488.
\newblock doi:{10.1007/BF00222330}.

\bibitem{Valdar2006c}
Valdar W, Flint J, Mott R.
\newblock {Simulating the collaborative cross: power of quantitative trait loci
  detection and mapping resolution in large sets of recombinant inbred strains
  of mice.}
\newblock Genetics. 2006;172(3):1783--97.
\newblock doi:{10.1534/genetics.104.039313}.

\bibitem{Aylor2011}
Aylor DL, Valdar W, Foulds-mathes W, Buus RJ, Verdugo RA, Baric RS, et~al.
\newblock {Genetic analysis of complex traits in the emerging Collaborative
  Cross}.
\newblock Genome Research. 2011;21:1213--22.
\newblock doi:{10.1101/gr.111310.110.mentally}.

\bibitem{Gralinski2015}
Gralinski LE, Ferris MT, Aylor DL, Whitmore AC, Green R, Frieman MB, et~al.
\newblock {Genome Wide Identification of SARS-CoV Susceptibility Loci Using the
  Collaborative Cross.}
\newblock PLoS Genetics. 2015;11(10):e1005504.
\newblock doi:{10.1371/journal.pgen.1005504}.

\bibitem{Kelada2016}
Kelada SNP.
\newblock {Plethysmography Phenotype QTL in Mice Before and After Allergen
  Sensitization and Challenge}.
\newblock G3 (Bethesda, Md). 2016;6(9):2857--2865.
\newblock doi:{10.1534/g3.116.032912}.

\bibitem{Donoghue2017}
Donoghue LJ, Livraghi-Butrico A, McFadden KM, Thomas JM, Chen G, Grubb BR,
  et~al.
\newblock {Identification of trans Protein QTL for Secreted Airway Mucins in
  Mice and a Causal Role for Bpifb1.}
\newblock Genetics. 2017;207(2):801--812.
\newblock doi:{10.1534/genetics.117.300211}.

\bibitem{Keele2019}
Keele GR, Crouse WL, Kelada SNP, Valdar W.
\newblock {Determinants of QTL Mapping Power in the Realized Collaborative
  Cross.}
\newblock G3 (Bethesda, Md). 2019;9(May):459966.
\newblock doi:{10.1534/g3.119.400194}.

\bibitem{King2012}
King EG, Merkes CM, McNeil CL, Hoofer SR, Sen S, Broman KW, et~al.
\newblock {Genetic dissection of a model complex trait using the Drosophila
  Synthetic Population Resource.}
\newblock Genome Research. 2012;22(8):1558--66.
\newblock doi:{10.1101/gr.134031.111}.

\bibitem{Dudbridge2004}
Dudbridge F, Koeleman BPC.
\newblock {Efficient Computation of Significance Levels for Multiple
  Associations in Large Studies of Correlated Data, Including Genomewide
  Association Studies}.
\newblock The American Journal of Human Genetics. 2004;75(3):424--435.
\newblock doi:{10.1086/423738}.

\bibitem{Doerge1996}
Doerge R, Churchill G.
\newblock {Permutation tests for multiple loci affecting a quantitative
  character}.
\newblock Genetics. 1996;142(1977):285--94.

\bibitem{Chesler2005}
Chesler EJ, Lu L, Shou S, Qu Y, Gu J, Wang J, et~al.
\newblock {Complex trait analysis of gene expression uncovers polygenic and
  pleiotropic networks that modulate nervous system function}.
\newblock Nature Genetics. 2005;37(3):233--242.
\newblock doi:{10.1038/ng1518}.

\bibitem{Wei2016}
Wei J, Xu S.
\newblock {A random-model approach to QTL mapping in multiparent advanced
  generation intercross (MAGIC) populations}.
\newblock Genetics. 2016;202(2):471--486.
\newblock doi:{10.1534/genetics.115.179945}.

\bibitem{Robinson1991}
Robinson GK.
\newblock {That BLUP is a Good Thing: The Estimation of Random Effects}.
\newblock Statistical Science. 1991;6(1):15--51.
\newblock doi:{10.2307/2246134}.

\bibitem{Baud2014}
Baud A, Guryev V, Hummel O, Johannesson M, {Rat Genome Sequencing and Mapping
  Consortium}, Flint J.
\newblock {Genomes and phenomes of a population of outbred rats and its
  progenitors}.
\newblock Scientific Data. 2014;1:140011.
\newblock doi:{10.1038/sdata.2014.11}.

\bibitem{Keele2018}
Keele GR, Prokop JW, He H, Holl K, Littrell J, Deal A, et~al.
\newblock {Genetic Fine-Mapping and Identification of Candidate Genes and
  Variants for Adiposity Traits in Outbred Rats}.
\newblock Obesity. 2018;26(1):213--222.
\newblock doi:{10.1002/oby.22075}.

\bibitem{Yalcin2005}
Yalcin B, Flint J, Mott R.
\newblock {Using progenitor strain information to identify quantitative trait
  nucleotides in outbred mice.}
\newblock Genetics. 2005;171(2):673--81.
\newblock doi:{10.1534/genetics.104.028902}.

\bibitem{Oreper2017}
Oreper D, Cai Y, Tarantino LM, de~Villena FPM, Valdar W.
\newblock {Inbred Strain Variant Database (ISVdb): A Repository for
  Probabilistically Informed Sequence Differences Among the Collaborative Cross
  Strains and Their Founders.}
\newblock G3 (Bethesda, Md). 2017;7(6):1623--1630.
\newblock doi:{10.1534/g3.117.041491}.

\bibitem{Lawrence2009}
Lawrence M, Gentleman R, Carey V.
\newblock rtracklayer: an R package for interfacing with genome browsers.
\newblock Bioinformatics. 2009;25:1841--1842.
\newblock doi:{10.1093/bioinformatics/btp328}.

\bibitem{Oreper2018}
Oreper D, Schoenrock SA, McMullan R, Ervin R, Farrington J, Miller DR, et~al.
\newblock {Reciprocal F1 Hybrids of Two Inbred Mouse Strains Reveal
  Parent-of-Origin and Perinatal Diet Effects on Behavior and Expression}.
\newblock G3 (Bethesda, Md). 2018;8(11):3447--3468.
\newblock doi:{10.1534/g3.118.200135}.

\bibitem{Keller2018}
Keller MP, Gatti DM, Schueler KL, Rabaglia ME, Stapleton DS, Simecek P, et~al.
\newblock {Genetic Drivers of Pancreatic Islet Function.}
\newblock Genetics. 2018;209(1):335--356.
\newblock doi:{10.1534/genetics.118.300864}.

\bibitem{RSoftware2019}
{R Core Team}. R: A Language and Environment for Statistical Computing; 2019.
\newblock Available from: \url{https://www.R-project.org/}.

\bibitem{Conway2017}
Conway JR, Lex A, Gehlenborg N.
\newblock {UpSetR: an R package for the visualization of intersecting sets and
  their properties}.
\newblock Bioinformatics. 2017;33(18):2938--2940.
\newblock doi:{10.1093/bioinformatics/btx364}.

\bibitem{Rieseberg1999}
Rieseberg LH, Archer MA, Wayne RK.
\newblock {Transgressive segregation, adaptation and speciation}.
\newblock Heredity. 1999;83(4):363--372.
\newblock doi:{10.1038/sj.hdy.6886170}.

\bibitem{Yang2011}
Yang H, Wang JR, Didion JP, Buus RJ, Bell TA, Welsh CE, et~al.
\newblock {Subspecific origin and haplotype diversity in the laboratory mouse.}
\newblock Nature Genetics. 2011;43(7):648--55.
\newblock doi:{10.1038/ng.847}.

\bibitem{Wang2012}
Wang JR, de Villena FPM, McMillan L.
\newblock {Comparative analysis and visualization of multiple collinear genomes.}
\newblock BMC Bioinformatics. 2012;13(S3):S13.
\newblock doi:{10.1186/1471-2105-13-S3-S13}.

\bibitem{Crowley2015}
Crowley JJ, Zhabotynsky V, Sun W, Huang S, Pakatci IK, Kim Y, et~al.
\newblock {Analyses of allele-specific gene expression in highly divergent
  mouse crosses identifies pervasive allelic imbalance.}
\newblock Nature Genetics. 2015;47(4):353--60.
\newblock doi:{10.1038/ng.3222}.

\bibitem{Braccini2015}
Braccini L, Ciraolo E, Campa CC, Perino A, Longo DL, Tibolla G, et~al.
\newblock {PI3K-C2$\gamma$ is a Rab5 effector selectively controlling endosomal
  Akt2 activation downstream of insulin signalling}.
\newblock Nature Communications. 2015;6(1):7400.
\newblock doi:{10.1038/ncomms8400}.

\bibitem{Gu2014}
Gu Z, Gu L, Eils R, Schlesner M, Brors B.
\newblock {circlize implements and enhances circular visualization in R}.
\newblock Bioinformatics. 2014;30(19):2811--2812.
\newblock doi:{10.1093/bioinformatics/btu393}.

\bibitem{HamiltonWilliams2010}
Hamilton-Williams EE, Wong SBJ, Martinez X, Rainbow DB, Hunter KM, Wicker LS,
  et~al.
\newblock {Idd9.2 and Idd9.3 Protective Alleles Function in CD4+ T-Cells and
  Nonlymphoid Cells to Prevent Expansion of Pathogenic Islet-Specific CD8+
  T-Cells}.
\newblock Diabetes. 2010;59(6):1478--1486.
\newblock doi:{10.2337/db09-1801}.

\bibitem{Ecco2017}
Ecco G, Imbeault M, Trono D.
\newblock {KRAB zinc finger proteins}.
\newblock Development. 2017;144(15):2719--2729.
\newblock doi:{10.1242/dev.132605}.

\bibitem{HamiltonWilliams2013}
Hamilton-Williams EE, Rainbow DB, Cheung J, Christensen M, Lyons PA, Peterson
  LB, et~al.
\newblock {Fine mapping of type 1 diabetes regions Idd9.1 and Idd9.2 reveals
  genetic complexity.}
\newblock Mammalian Genome. 2013;24(9-10):358--75.
\newblock doi:{10.1007/s00335-013-9466-y}.

\bibitem{Richard2012}
Richards AL, Jones L, Moskvina V, Kirov G, Gejman PV, Levinson DF, et~al.
\newblock {Schizophrenia susceptibility alleles are enriched for alleles that affect gene expression in adult human brain.}
\newblock Molecular Psychiatry. 2012;17(2):193--201.
\newblock doi:{10.1038/mp.2011.11}.

\bibitem{Bryois2018}
Bryois J, Garrett ME, Song L, Safi A, Giusti-Rodriguez P, Johnson GD, et~al.
\newblock {Evaluation of chromatin accessibility in prefrontal cortex of individuals with schizophrenia.}
\newblock Nature Communications. 2018;9(12):3121.
\newblock doi:{10.1038/s41467-018-05379-y}.

\bibitem{GTEX2017}
Aguet F, Brown AA, Castel SE, Davis JR, He Y, Jo B, et~al.
\newblock {Genetic effects on gene expression across human tissues}.
\newblock Nature. 2017;550(7675):204--213.
\newblock doi:{10.1038/nature24277}.

\bibitem{Fagny2017}
Fagny M, Paulson JN, Kuijjer ML, Sonawane AR, Chen C, Lopes-Ramos CM, et~al.
\newblock {Exploring regulation in tissues with eQTL networks}.
\newblock Proceedings of the National Academy of Sciences. 2017;114(9):7841--50.
\newblock doi:{10.1038/nature24277}.

\bibitem{Huang2009}
Huang GJ, Shifman S, Valdar W, Johannesson M, Yalcin B, Taylor MS, et~al.
\newblock {High resolution mapping of expression QTLs in heterogeneous stock
  mice in multiple tissues}.
\newblock Genome Research. 2009;19(6):1133--1140.
\newblock doi:{10.1101/gr.088120.108}.

\bibitem{Sul2013}
Sul JH, Han B, Ye C, Choi T, Eskin E.
\newblock {Effectively Identifying eQTLs from Multiple Tissues by Combining
  Mixed Model and Meta-analytic Approaches}.
\newblock PLoS Genetics. 2013;9(6):e1003491.
\newblock doi:{10.1371/journal.pgen.1003491}.

\bibitem{Fu2012a}
Fu J, Wolfs MGM, Deelen P, Westra HJ, Fehrmann RSN, {Te Meerman} GJ, et~al.
\newblock {Unraveling the regulatory mechanisms underlying tissue-dependent
  genetic variation of gene expression.}
\newblock PLoS Genetics. 2012;8(1):e1002431.
\newblock doi:{10.1371/journal.pgen.1002431}.

\bibitem{Flutre2013}
Flutre T, Wen X, Pritchard J, Stephens M.
\newblock {A statistical framework for joint eQTL analysis in multiple
  tissues.}
\newblock PLoS Genetics. 2013;9(5):e1003486.
\newblock doi:{10.1371/journal.pgen.1003486}.

\bibitem{Acharya2016}
Acharya CR, McCarthy JM, Owzar K, Allen AS.
\newblock {Exploiting expression patterns across multiple tissues to map
  expression quantitative trait loci}.
\newblock BMC Bioinformatics. 2016;17(1):257.
\newblock doi:{10.1186/s12859-016-1123-5}.

\bibitem{Rutledge2014}
Rutledge H, Aylor DL, Carpenter DE, Peck BC, Chines P, Ostrowski LE, et~al.
\newblock {Genetic regulation of Zfp30, CXCL1, and neutrophilic inflammation in
  murine lung.}
\newblock Genetics. 2014;198(2):735--45.
\newblock doi:{10.1534/genetics.114.168138}.

\bibitem{Kelada2014}
Kelada SNP, Carpenter DE, Aylor DL, Chines P, Rutledge H, Chesler EJ, et~al.
\newblock {Integrative genetic analysis of allergic inflammation in the murine
  lung.}
\newblock American Journal of Respiratory Cell and Molecular Giology.
  2014;51(3):436--45.
\newblock doi:{10.1165/rcmb.2013-0501OC}.

\bibitem{Xu2003}
Xu S.
\newblock {Theoretical basis of the Beavis effect.}
\newblock Genetics. 2003;165(4):2259--68.
\newblock doi:{10.1534/genetics.114.161968}.

\end{thebibliography}

\newpage

\setcounter{table}{0}
\setcounter{figure}{0}


\begin{figure}[hp]
\includegraphics[width=\linewidth, clip, trim={0in 0.1in 0in 0.2in}]{figs/overview_diagram_simplified.png}
\caption{}
\end{figure}

\newpage

\begin{figure}[hp]
\includegraphics[width=\textwidth, trim={0in 0in 0in 0in}, clip]{figs/mapping_results_revision.png}
\caption{}
\end{figure}

\newpage

\begin{figure}[hp]
\includegraphics[width=\textwidth, trim={0in 0in 0in 0in}, clip]{figs/correlated_local_eqtl.png}
\caption{}
\end{figure}

\newpage

\begin{figure}[hp]
\includegraphics[width=\textwidth, trim={0in 0.5in 0in 0in}, clip]{figs/pik3c2g_example.png}
\caption{}
\end{figure}

\newpage

\begin{figure}[hp]
\includegraphics[width=\textwidth, trim={0in 0in 0in 0in}, clip]{figs/correlated_distal_eqtl.png}
\caption{}
\end{figure}

\newpage

\begin{figure}[hp]
\includegraphics[width=\linewidth, clip, trim={0in 0in 0in 0in}]{figs/mediation_graph.png}
\caption{}
\end{figure}


\begin{figure}[hp]
\includegraphics[width=\textwidth, trim={0in 0in 0in 0in}, clip]{figs/ccdc137_mediation.png}
\caption{}
\end{figure}

\newpage

\begin{figure}[hp]
\includegraphics[width=\textwidth, trim={0in 0in 0in 0in}, clip]{figs/circos_over_tissues.png}
\caption{}
\end{figure}

\newpage

\begin{figure}[hp]
\includegraphics[width=\textwidth, trim={0in 0.1in 0in 0in}, clip]{figs/akr1e1_full_model_update.png}
\caption{}
\end{figure}

\end{document}

